%% Combined references

\begin{thebibliography}{9}

\bibitem[Afgan 2018]{Afgan 2018} Enis Afgan, Dannon Baker, Bérénice Batut, Marius van
den Beek, Dave Bouvier, Martin Čech, John Chilton, Dave Clements, Nate
Coraor, Björn A Grüning, Aysam Guerler, Jennifer Hillman-Jackson, Saskia
Hiltemann, Vahid Jalili, Helena Rasche, Nicola Soranzo, Jeremy Goecks,
James Taylor, Anton Nekrutenko, Daniel Blankenberg (2018):\\
\textbf{The Galaxy platform for accessible, reproducible and
collaborative biomedical analyses: 2018 update}.\\
\emph{Nucleic Acids Research} \textbf{46}(W1) W537--W544\\
\url{https://doi.org/10.1093/nar/gky379}

\bibitem[Agarwal 2021]{Agarwal 2021}~Deborah~Agarwal, Carole~Goble, Stian~Soiland-Reyes,
Ugis~Sarkans, Daniel~Noesgaard, Uwe~Schindler, Martin~Fenner,
Paolo~Manghi, Shelley~Stall, Caroline~Coward, Chris~Erdmann (2021):\\
\textbf{Data Citation Community of Practice -- 8 June 2021 Workshop}.\\
\emph{Zenodo/AGU}\\
\url{https://data.agu.org/DataCitationCoP/2nd-workshop-data-citation}~\\
\url{https://doi.org/10.5281/zenodo.4916734}

\bibitem[Almeida 2019]{Almeida 2019}~Alexandre Almeida, Alex L. Mitchell, Miguel Boland, Samuel C.
Forster, Gregory B. Gloor, Aleksandra Tarkowska, Trevor D. Lawley,
Robert D. Finn (2019):\\
\textbf{A new genomic blueprint of the human gut microbiota}.\\
\emph{Nature} \textbf{568}(7753) 499--504.\\
\url{https://doi.org/10.1038/s41586-019-0965-1}

\bibitem[Alterovitz 2018]{Alterovitz 2018} Gil Alterovitz, Dennis A Dean II, Carole Goble, Michael R
Crusoe, Stian Soiland-Reyes, Amanda Bell, Anais Hayes, Anita Suresh,
Charles Hadley S King IV, Dan Taylor, KanakaDurga Addepalli, Elaine
Johanson, Elaine E Thompson, Eric Donaldson, Hiroki Morizono, Hsinyi
Tsang, Jeet K Vora, Jeremy Goecks, Jianchao Yao, Jonas S Almeida,
Jonathon Keeney, KanakaDurga Addepalli, Konstantinos Krampis, Krista
Smith, Lydia Guo, Mark Walderhaug, Marco Schito, Matthew Ezewudo, Nuria
Guimera, Paul Walsh, Robel Kahsay, Srikanth Gottipati, Timothy C
Rodwell, Toby Bloom, Yuching Lai, Vahan Simonyan, Raja Mazumder
(2018):\\
\textbf{Enabling precision medicine via standard communication of HTS
provenance, analysis, and results}.\\
\emph{PLOS Biology} \textbf{16}(12):e3000099\\
\url{https://doi.org/10.1371/journal.pbio.3000099}

\bibitem[Alves 2022]{Alves 2022} Renato Alves, Dimitrios Bampalikis, Leyla Jael Castro,
José María Fernández, Jennifer Harrow, Mateusz Kuzak, Eva Martin, Fotis
E. Psomopoulos, Allegra Via (2022):\\
\textbf{ELIXIR Software Management Plan for Life Sciences}.\\
\emph{BioHackrXiv}\\
\url{https://doi.org/10.37044/osf.io/k8znb}

\bibitem[Amorim 2016]{Amorim 2016}~Ricardo Carvalho Amorim, João Aguiar Castro, João Rocha da
Silva, Cristina Ribeiro (2016):\\
\textbf{A comparison of research data management platforms:
Architecture, flexible metadata and interoperability}.\\
\emph{Universal Access in the Information Society} \textbf{16} pp
851--862.\\
\url{https://doi.org/10.1007/s10209-016-0475-y}

\bibitem[Arfaoui 2020]{Arfaoui 2020}~Ghaith Arfaoui, Maroua Jaoua (2020):\\
\textbf{RO-Crate RDA maDMP Mapper}.\\
\emph{Zenodo}
\url{https://github.com/GhaithArf/ro-crate-rda-madmp-mapper}\\
\url{https://doi.org/10.5281/zenodo.3922136}

\bibitem[Bacall 2019]{Bacall 2019}~Finn Bacall, Stian~Soiland-Reyes, Marina~Soares e Silva
(2019):\\
\textbf{eScienceLab: RO-Composer}.\\
\url{https://esciencelab.org.uk/projects/ro-composer/}~\\
\url{https://github.com/ResearchObject/research-object-composer}

\bibitem[Bacall 2022]{Bacall 2022} Finn Bacall, Alan R. Williams, Stuart Owen, Stian
Soiland-Reyes (2022):\\
\textbf{Workflow RO-Crate Profile 1.0.}\\
\url{https://w3id.org/workflowhub/workflow-ro-crate/1.0}

\bibitem[Bacall 2022b]{Bacall 2022b}~Finn Bacall, Martyn~Whitwell (2022):\\
\textbf{GitHub -- ResearchObject/ro-crate-ruby: A Ruby gem for creating,
manipulating and reading RO-Crates}.\\
\url{https://github.com/ResearchObject/ro-crate-ruby}

\bibitem[Bayarri 2022]{Bayarri 2022} Genís Bayarri, Adam Hospital (2022): \textbf{CWL GMX
Automatic Ligand Parameterization tutorial}.\\
\emph{Workflow Hub} (Common Workflow Language)\\
\url{https://doi.org/10.48546/workflowhub.workflow.255.1}

\bibitem[Baker 2020]{Baker 2020} Dannon Baker, Marius van den Beek, Daniel Blankenberg, Dave
Bouvier, John Chilton, Nate Coraor, Frederik Coppens, Ignacio Eguinoa,
Simon Gladman, Björn Grüning, Nicholas Keener, Delphine Larivière,
Andrew Lonie, Sergei Kosakovsky Pond, Wolfgang Maier, Anton Nekrutenko,
James Taylor, Steven Weaver (2020):\\
\textbf{No more business as usual: Agile and effective responses to
emerging pathogen threats require open data and open analytics}.\\
\emph{PLOS Pathogens} \textbf{16}(8):e1008643.\\
\url{https://doi.org/10.1371/journal.ppat.1008643}

\bibitem[Barker 2019]{Barker 2019}~Michelle Barker, Ross Wilkinson, Andrew Treloar (2019):\\
\textbf{The Australian Research Data Commons}.\\
\emph{Data Science Journal} \textbf{18} (2019).\\
\url{https://doi.org/10.5334/dsj-2019-044}

\bibitem[Bechhofer 2013]{Bechhofer 2013}~Sean Bechhofer, Iain Buchan, David De Roure, Paolo Missier,
John Ainsworth, Jiten Bhagat, Phillip Couch, Don Cruickshank, Mark
Delderfield, Ian Dunlop, Matthew Gamble, Danius Michaelides, Stuart
Owen, David Newman, Shoaib Sufi, Carole Goble (2013):\\
\textbf{Why Linked Data is not enough for scientists}.\\
\emph{Future Generation Computer Systems} \textbf{29}(2),
pp.~599--611.\\
\url{https://doi.org/10.1016/j.future.2011.08.004}

\bibitem[Belchev 2021]{Belchev 2021}~Kostadin~Belchev (2021):\\
\textbf{KockataEPich/CheckMyCrate: A command line application for
validating a RO-Crate object against a JSON profile}.\\
\emph{GitHub}.\\
\url{https://github.com/KockataEPich/CheckMyCrate}

\bibitem[Belhajjame 2015]{Belhajjame 2015}~Khalid Belhajjame, Jun Zhao, Daniel Garijo, Matthew Gamble,
Kristina Hettne, Raul Palma, Eleni Mina, Oscar Corcho, José Manuel
Gómez-Pérez, Sean Bechhofer, Graham Klyne, Carole Goble (2015):\\
\textbf{Using a suite of ontologies for preserving workflow-centric
research objects}.\\
\emph{Web Semantics: Science, Services and Agents on the World Wide Web}
\textbf{32} pp.~16--42.\\
\url{https://doi.org/10.1016/j.websem.2015.01.003}

\bibitem[Benureau 2017]{Benureau 2017}~Fabien C. Y. Benureau, Nicolas P. Rougier (2017):\\
\textbf{Re-run, repeat, reproduce, reuse, replicate: Transforming code
into scientific contributions}.\\
\emph{Frontiers in Neuroinformatics} \textbf{11}:69.\\
\url{https://doi.org/10.3389/fninf.2017.00069}

\bibitem[Berman 2007]{Berman 2007}~Helen Berman, Kim Henrick, Haruki Nakamura, John L Markley
(2007):\\
\textbf{The worldwide Protein Data Bank (wwPDB): Ensuring a single,
uniform archive of PDB data}.\\
\emph{Nucleic Acids Research} \textbf{35}(Database issue) ,
D301--D303.\\
\url{https://doi.org/10.1093/nar/gkl971}

\bibitem[Bietrix 2021]{Bietrix 2021}~Florence Bietrix, José Maria Carazo, Salvador
Capella-Gutierrez, Frederik Coppens, Maria Luisa Chiusano, Romain David,
Jose Maria Fernandez, Maddalena Fratelli, Jean-Karim Heriche, Carole
Goble, Philip Gribbon, Petr Holub, Robbie Joosten, Simone Leo, Stuart
Owen, Helen Parkinson, Roland Pieruschka, Luca Pireddu, Luca Porcu,
Michael Raess, Laura Rodriguez-Navas, Andreas Scherer, Stian
Soiland-Reyes, Jing Tang (2021):\\
\textbf{EOSC-life methodology framework to enhance reproducibility
within EOSC-life}.\\
\emph{Zenodo}\\
\url{https://doi.org/10.5281/zenodo.4705078}

\bibitem[Bizer 2009]{Bizer 2009} C Bizer, Tom Heath, Tim Berners-Lee (2009):\\
\textbf{Linked Data - The Story So Far}.\\
\emph{International Journal on Semantic Web and Information Systems}
\textbf{5}(3)\\
\url{https://doi.org/10.4018/jswis.2009081901}

\bibitem[Bizer 2011]{Bizer 2011}~Christian Bizer, Tom Heath, Tim Berners-Lee (2011):\\
\textbf{Linked data: The story so far}.\\
In \emph{Semantic Services, Interoperability and Web Applications:
Emerging Concepts}, Amit Sheth (ed.) ISBN 9781609605933\\
\url{https://doi.org/10.4018/978-1-60960-593-3.ch008}

\bibitem[Bonino 2019]{bonino2019}
Luiz Bonino, Peter Wittenburg, Bonnie Carroll, Alex
Hardisty, Mark Leggott, Carlo Zwölf (2019):\\
\textbf{FAIR digital object framework v1.02. \emph{FDOF technical
implementation guideline}}\\
\url{https://github.com/GEDE-RDA-Europe/GEDE/blob/master/FAIR\%20Digital\%20Objects/FDOF/FAIR\%20Digital\%20Object\%20Framework-v1-02.docx}

\bibitem[Bonino 2021]{bonino2021} Luiz Olavo Bonino da Silva Santos (2021):\\
\textbf{FAIR Digital Object Framework Documentation}.\\
\emph{Working Draft}\\
\url{https://fairdigitalobjectframework.org/}

\bibitem[Brack 2022]{Brack 2022} Paul Brack, Oliver Woolland, Laurence Livermore
(2022):\\
\textbf{De novo digitisation.} (Galaxy workflow) \emph{WorkflowHub}\\
\url{https://doi.org/10.48546/workflowhub.workflow.373.1}

\bibitem[Brand 2015]{Brand 2015} Amy Brand, Liz Allen, Micah Altman, Marjorie Hlava, Jo Scott
(2015):\\
\textbf{Beyond authorship: Attribution, contribution, collaboration, and
credit}.\\
\emph{Learned Publishing} \textbf{28}(2) pp.~151--155.\\
\url{https://doi.org/10.1087/20150211}

\bibitem[Brenner 2020]{Brenner 2020} Gabriel~Brenner (2020):\\
\textbf{BrennerG/Ro-Crate\_2\_ma-DMP: v1.0.0}.\\
\url{https://github.com/BrennerG/Ro-Crate_2_ma-DMP}~\\
\url{https://doi.org/10.5281/zenodo.3903463}

\bibitem[Cardoso 2020a]{Cardoso 2020a}~João Cardoso, Diogo Proença, José Borbinha (2020):\\
\textbf{Machine-actionable data management plans: A knowledge retrieval
approach to automate the assessment of funders' requirements}.\\
\emph{ECIR 2020: Advances in Information Retrieval}\\
ISBN 978-3-030-45442-5.\\
\url{https://doi.org/10.1007/978-3-030-45442-5_15}


\bibitem[Cardoso 2020b]{Cardoso 2020b} João Cardoso, Leyla Jael Garcia Castro, Fajar Ekaputra, Marie-Christine Jacquemot-Perbal, Tomasz Miksa, José Borbinha (2020):\\
\textbf{Towards Semantic Representation of Machine-Actionable Data
Management Plans}.\\
\emph{PUBLISSO}.\\
\url{https://repository.publisso.de/resource/frl:6423289}~\\
\url{https://doi.org/10.4126/frl01-006423289}

\bibitem[Ó Carragáin 2019]{Ó Carragáin 2019}~Eoghan~Ó~Carragáin, Carole~Goble, Peter Sefton,
Stian~Soiland-Reyes (2019):\\
\textbf{A lightweight approach to research object data packaging}.\\
\emph{Bioinformatics Open Source Conference (BOSC2019)},
2019-07-24/2019-07-25, Basel, Switzerland.\\
\emph{Zenodo}.\\
\url{https://doi.org/10.5281/zenodo.3250687}

\bibitem[Chard 2016]{Chard 2016}~Kyle Chard, Mike D' Arcy, Ben Heavner, Ian Foster, Carl
Kesselman, Ravi Madduri, Alexis Rodriguez, Stian Soiland-Reyes, Carole
Goble, Kristi Clark, Eric W. Deutsch, Ivo Dinov, Nathan Price, Arthur
Toga (2016):\\
\textbf{I'll take that to go: Big data bags and minimal identifiers for
exchange of large, complex datasets}.\\
\emph{2016 IEEE International Conference on Big Data (Big Data)}, IEEE,
pp.~319--328.\\
ISBN 978-1-4673-9005-7.\\
\url{https://static.aminer.org/pdf/fa/bigdata2016/BigD418.pdf}~\\
\url{https://doi.org/10.1109/BigData.2016.7840618}

\bibitem[Chard 2019]{Chard 2019}~Kyle Chard, Niall Gaffney, Matthew B. Jones, Kacper Kowalik,
Bertram Ludascher, Timothy McPhillips, Jarek Nabrzyski, Victoria
Stodden, Ian Taylor, Thomas Thelen, Matthew J. Turk, Craig Willis
(2019):\\
\textbf{Application of BagIt-serialized research object bundles for
packaging and re-execution of computational analyses}.\\
\emph{15th International Conference on eScience (eScience 2019)}, IEEE,
pp.~514--521.\\
ISBN 978-1-7281-2451-3.\\
\url{https://zenodo.org/record/3381754}~\\
\url{https://doi.org/10.1109/eScience.2019.00068}

\bibitem[Chard 2014]{Chard 2014}~Kyle~Chard, Steven~Tuecke and Ian~Foster (2014):\\
\textbf{Efficient and secure transfer, synchronization, and sharing of
big data}.\\
\emph{IEEE Cloud Computing} \textbf{1}(3) pp.~46--55.\\
\url{https://doi.org/10.1109/MCC.2014.52}

\bibitem[Ciccarese 2017]{Ciccarese 2017}~Paolo Ciccarese, Robert Sanderson, Benjamin Young (2017):\\
\textbf{Web Annotation Data Model}.\\
\emph{W3C Recommendation} 23 February 2017.
\url{https://www.w3.org/TR/2017/REC-annotation-model-20170223/}

\bibitem[Claerbout 1992]{Claerbout 1992}~Jon F. Claerbout, Martin Karrenbach (1992):\\
\textbf{Electronic documents give reproducible research a new
meaning}.\\
\emph{SEG Technical Program Expanded Abstracts 1992}, Society of
Exploration Geophysicists, pp.~601--604.\\
\url{http://sep.stanford.edu/oldsep/matt/join/redoc/web/seg92.html}\\
\url{https://doi.org/10.1190/1.1822162}

\bibitem[Cohen-Boulakia 2017]{Cohen-Boulakia 2017}~Sarah Cohen-Boulakia, Khalid Belhajjame, Olivier Collin, Jérôme
Chopard, Christine Froidevaux, Alban Gaignard, Konrad Hinsen, Pierre
Larmande, Yvan Le Bras, Frédéric Lemoine, Fabien Mareuil, Hervé Ménager,
Christophe Pradal, Christophe Blanchet (2017):\\
\textbf{Scientific workflows for computational reproducibility in the
life sciences: Status, challenges and opportunities}.\\
\emph{Future Generation Computer Systems} \textbf{75} pp.~284--298.\\
\url{https://hal.archives-ouvertes.fr/hal-01516082}\\
\url{https://doi.org/10.1016/j.future.2017.01.012}

\bibitem[Cossu 2018]{Cossu 2018}~Stefano Cossu, Esmé Cowles, Karen Estlund, Christina Harlow,
Tom Johnson, Mark Matienzo, Danny Lamb, Lynette Rayle, Rob Sanderson,
Jon Stroop, Andrew Woods (2018):\\
\textbf{Portland Common Data Model}.\\
\emph{GitHub duraspace/pcdm Wiki} (2018-06-15)
\url{https://github.com/duraspace/pcdm/wiki}

\bibitem[Crosas 2011]{Crosas 2011}~Mercè~Crosas (2011):\\
\textbf{The DataVerse Network: An open-source application for sharing,
discovering and preserving data}.\\
\emph{D-Lib Magazine} \textbf{17}(1/2)a\\
\url{https://doi.org/10.1045/january2011-crosas}

\bibitem[Crosas 2020]{Crosas 2020}~Mercè~Crosas (2020):\\
\textbf{Harvard Data Commons}.\\
\emph{European Dataverse Workshop 2020}, Tromsø, Norway. ISSN
2387-3086.\\
\url{https://doi.org/10.7557/5.5422}

\bibitem[Crosswell 2012]{Crosswell 2012}~Lindsey C Crosswell, Janet M Thornton (2012):\\
\textbf{ELIXIR: A distributed infrastructure for European biological
data}.\\
\emph{Trends in Biotechnology} \textbf{30}(5) pp.~241--242.\\
\url{https://doi.org/10.1016/j.tibtech.2012.02.002}

\bibitem[CRS4 2022]{CRS4 2022}~CRS4 (2022):\\
\textbf{LifeMonitor, a testing and monitoring service for scientific
workflows}.\\
\url{https://about.lifemonitor.eu/}

\bibitem[Crusoe 2022]{Crusoe 2022} Michael R. Crusoe, Sanne Abeln, Alexandru Iosup, Peter
Amstutz, John Chilton, Nebojša Tijanić, Hervé Ménager, Stian
Soiland-Reyes, Bogdan Gavrilović, Carole Goble, The CWL Community
(2022):\\
\textbf{Methods Included: Standardizing
Computational Reuse and Portability with the Common Workflow
Language}.\\
\emph{Communications of the ACM} \textbf{65}(6)\\
\url{https://arxiv.org/abs/2105.07028}\\
\url{https://doi.org/10.1145/3486897}

\bibitem[DCAT2 2020]{DCAT2 2020}~Riccardo~Albertoni, David~Browning, Simon~Cox,
Alejandra~Gonzalez Beltran, Andrea~Perego, Peter~Winstanley, Dataset
Exchange Working Group (2020):\\
\textbf{Data Catalog Vocabulary (DCAT) -- Version 2}.\\
\emph{W3C Recommendation} (2020)\\
\url{https://www.w3.org/TR/2020/REC-vocab-dcat-2-20200204/}


\bibitem[De Geest 2022]{De Geest 2022} Paul De Geest, Frederik Coppens, Stian
Soiland-Reyes, Ignacio Eguinoa, Simone Leo (2022):\\
\textbf{Enhancing RDM in Galaxy by integrating RO-Crate}.\\
1st International Conference on FAIR Digital Objects
(\href{https://www.fdo2022.org/}{FDO 2022}) (poster)\\
\emph{Research Ideas and Outcomes} (accepted)

\bibitem[de Mello 2022]{de Mello 2022} Blanda Helena de Mello, Sandro José Rigo, Cristiano
André da Costa, Rodrigo da Rosa Righi, Bruna Donida, Marta Rosecler Bez,
Luana Carina Schunke (2022):\\
\textbf{Semantic interoperability in health records standards: a
systematic literature review}.\\
\emph{Health and Technology} \textbf{12}\\
\url{https://doi.org/10.1007/s12553-022-00639-w}

\bibitem[DONA 2018]{DONA 2018} DONA Foundation (2018):\\
\textbf{Digital Object Interface Protocol specification, version 2.0}.\\
\url{https://hdl.handle.net/0.DOIP/DOIPV2.0}

\bibitem[De Smedt 2020]{De Smedt 2020} Koenraad De Smedt, Dimitris Koureas, Peter
Wittenburg (2020):\\
\textbf{FAIR Digital Objects for Science: From Data Pieces to Actionable
Knowledge Units}.\\
\emph{Publications)} \textbf{8}(2):21\\
\url{https://doi.org/10.3390/publications8020021}




\bibitem[Galaxy 2022]{Galaxy 2022} The Galaxy Community (E. Afgan, A. Nekrutenko, B. A.
Grüning, D. Blankenberg, J. Goecks, M. C. Schatz, A. E. Ostrovsky, A.
Mahmoud, A. J. Lonie, A. Syme, A. Fouilloux, A. Bretaudeau, A.
Nekrutenko, A. Kumar, A. C. Eschenlauer, A. D. DeSanto, A. Guerler, B.
Serrano-Solano, B. Batut, B. A. Grüning, B. W. Langhorst, B. Carr, B. A.
Raubenolt, C. J. Hyde, C. J. Bromhead, C. B. Barnett, C. Royaux, C.
Gallardo, D. Blankenberg, D. J. Fornika, D. Baker, D. Bouvier, D.
Clements, D. A. de Lima Morais, D. L. Tabernero, D. Lariviere, E. Nasr,
E. Afgan, F. Zambelli, F. Heyl, F. Psomopoulos, F. Coppens, G. R. Price,
G. Cuccuru, G. L. Corguillé, G. Von Kuster, G. G. Akbulut, H. Rasche, H.
Hans-Rudolf, I. Eguinoa, I. Makunin, I. J. Ranawaka, J. P. Taylor, J.
Joshi, J. Hillman-Jackson, J. Goecks, J. M. Chilton, K. Kamali, K.
Suderman, K. Poterlowicz, L. B. Yvan, L. Lopez-Delisle, L. Sargent, M.
E. Bassetti, M. A. Tangaro, M. van den Beek, M. Čech, M. Bernt, M.
Fahrner, M. Tekman, M. C. Föll, M. C. Schatz, M. R. Crusoe, M.
Roncoroni, N. Kucher, N. Coraor, N. Stoler, N. Rhodes, N. Soranzo, N.
Pinter, N. A. Goonasekera, P. A. Moreno, P. Videm, P. Melanie, P.
Mandreoli, P. D. Jagtap, Q. Gu, R. J. M. Weber, R. Lazarus, R. H. P.
Vorderman, S. Hiltemann, S. Golitsynskiy, S. Garg, S. A. Bray, S. L.
Gladman, S. Leo, S. P. Mehta, T. J. Griffin, V. Jalili, V. Yves, V. Wen,
V. K. Nagampalli, W. A. Bacon, W. de Koning, W. Maier, P. J. Briggs)
(2022):\\
\textbf{The Galaxy platform for accessible, reproducible and
collaborative biomedical analyses: 2022 update}.\\
\emph{Nucleic Acids Research} \textbf{50}\\
\url{https://doi.org/10.1093/nar/gkac247}

\bibitem[Goble 2021]{Goble 2021} Carole Goble, Stian Soiland-Reyes, Finn Bacall, Stuart
Owen, Alan Williams, Ignacio Eguinoa, Bert Droesbeke, Simone Leo, Luca
Pireddu, Laura Rodríguez-Navas, José Mª Fernández, Salvador
Capella-Gutierrez, Hervé Ménager, Björn Grüning, Beatriz Serrano-Solano,
Philip Ewels, Frederik Coppens (2021):\\
\textbf{Implementing FAIR Digital Objects in the EOSC-Life Workflow
Collaboratory}.\\
\emph{Zenodo}\\
\url{https://doi.org/10.5281/zenodo.4605654}

\bibitem[Guha 2016]{Guha 2016} R. V. Guha, Dan Brickley, Steve Macbeth (2016):\\
\textbf{Schema.org: evolution of structured data on the web}.\\
\emph{Communications of the ACM} \textbf{59}(2)\\
\url{https://doi.org/10.1145/2844544}

\bibitem[Hardisty 2019]{Hardisty 2019} Alex R Hardisty, Keping Ma, Gil Nelson, Jose Fortes
(2019):\\
\textbf{`openDS' -- A New Standard for Digital Specimens and Other
Natural Science Digital Object Types}.\\
\emph{Biodiversity Information Science and Standards}
\textbf{3}:e37033\\
\url{https://doi.org/10.3897/biss.3.37033}

\bibitem[Hardisty 2022]{Hardisty 2022} Alex Hardisty, Paul Brack, Carole Goble, Laurence
Livermore, Ben Scott, Quentin Groom, Stuart Owen, Stian Soiland-Reyes
(2022):\\
\href{../specimen-data-refinery/}{\textbf{The Specimen Data Refinery: A
canonical workflow framework and FAIR Digital Object approach to
speeding up digital mobilisation of natural history collections}}.\\
\emph{Data Intelligence} \textbf{4}(2)\\
\url{https://doi.org/10.1162/dint_a_00134}

\bibitem[Hasnain 2018]{Hasnain 2018} Ali Hasnain, Dietrich Rebholz-Schuhmann (2019):\\
\textbf{Assessing FAIR Data Principles Against the 5-Star Open Data
Principles}.\\
ESWC 2018: The Semantic Web: ESWC 2018 Satellite Events,\\
\emph{Lecture Notes in Computer Science} \textbf{11155}\\
\url{https://doi.org/10.1007/978-3-319-98192-5_60}

\bibitem[Jacobsen 2020]{Jacobsen 2020} Annika Jacobsen, Ricardo de Miranda Azevedo, Nick
Juty, Dominique Batista, Simon Coles, Ronald Cornet, Mélanie Courtot,
Mercè Crosas, Michel Dumontier, Chris T. Evelo, Carole Goble, Giancarlo
Guizzardi, Karsten Kryger Hansen, Ali Hasnain, Kristina Hettne, Jaap
Heringa, Rob W.W. Hooft, Melanie Imming, Keith G. Jeffery, Rajaram
Kaliyaperumal, Martijn G. Kersloot, Christine R. Kirkpatrick, Tobias
Kuhn, Ignasi Labastida, Barbara Magagna, Peter McQuilton, Natalie
Meyers, Annalisa Montesanti, Mirjam van Reisen, Philippe Rocca-Serra,
Robert Pergl, Susanna-Assunta Sansone, Luiz Olavo Bonino da Silva
Santos, Juliane Schneider, George Strawn, Mark Thompson, Andra
Waagmeester, Tobias Weigel, Mark D. Wilkinson, Egon L. Willighagen,
Peter Wittenburg, Marco Roos, Barend Mons, Erik Schultes (2020):\\
\textbf{FAIR Principles: Interpretations and Implementation
Considerations}.\\
\emph{Data Intelligence} \textbf{2}(1):10--29\\
\url{https://doi.org/10.1162/dint_r_00024}

\bibitem[Kuhn 2021]{Kuhn 2021} Tobias Kuhn, Vincent Emonet, Haris Antonatos, Stian
Soiland-Reyes, Michel Dumontier (2021):\\
\href{../../../2021/phd/nanopub/}{\textbf{Semantic micro-contributions
with decentralized nanopublication services}}.\\
\emph{PeerJ Computer Science} \textbf{7}:e387\\
\url{https://doi.org/10.7717/peerj-cs.387}

\bibitem[Livermore 2022a]{Livermore 2022a} Laurence Livermore, Oliver Woolland (2022):\\
\textbf{DLA-Collections-test.} (Galaxy workflow)\\
\emph{WorkflowHub}
\url{https://doi.org/10.48546/workflowhub.workflow.374.1}

\bibitem[Livermore 2022b]{Livermore 2022b} Laurence Livermore, Oliver Woolland (2022):\\
\textbf{HTR-Collections-test}. (Galaxy workflow)\\
\emph{WorkflowHub}\\
\url{https://doi.org/10.48546/workflowhub.workflow.375.1}

\bibitem[Mai Chan 1995]{Mai Chan 1995} Lois Mai~Chan (1995):\\
\textbf{Library of Congress Subject Headings: Principles and
Application}, 3rd edn, p.~556.\\
\href{https://identifiers.org/isbn/9781563081910}{ISBN 9781563081910}.

\bibitem[McMurry 2017]{McMurry 2017} Julie A McMurry, Nick Juty, Niklas Blomberg, Tony
Burdett, Tom Conlin, Nathalie Conte, Mélanie Courtot, John Deck, Michel
Dumontier, Donal K Fellows, Alejandra Gonzalez-Beltran, Philipp
Gormanns, Jeffrey Grethe, Janna Hastings, Jean-Karim Hériché, Henning
Hermjakob, Jon C Ison, Rafael C Jimenez, Simon Jupp, John Kunze, Camille
Laibe, Nicolas Le Novère, James Malone, Maria Jesus Martin, Johanna R
McEntyre, Chris Morris, Juha Muilu, Wolfgang Müller, Philippe
Rocca-Serra, Susanna-Assunta Sansone, Murat Sariyar, Jacky L Snoep,
Stian Soiland-Reyes, Natalie J Stanford, Neil Swainston, Nicole
Washington, Alan R Williams, Sarala M Wimalaratne, Lilly M Winfree,
Katherine Wolstencroft, Carole Goble, Cristopher J Mungall, Melissa A
Haendel, Helen Parkinson (2017):\\
\textbf{Identifiers for the 21st century: How to design, provision, and
reuse identifiers to maximize utility and impact of life science
data.}\\
\emph{PLOS Biology} \textbf{15}(6):e2001414
\url{https://doi.org/10.1371/journal.pbio.2001414}

\bibitem[Miksa 2020]{Miksa 2020} Tomasz Miksa, Maroua Jaoua, Ghaith Arfaoui (2020):\\
\textbf{Research Object Crates and Machine-actionable Data Management
Plans}.\\
\emph{First Workshop on Data and Research Objects Management for Linked
Open Science (DaMaLOS)} at The 19th International Semantic Web
Conference (ISWC 2020).\\
\url{https://doi.org/10.4126/frl01-006423291}


\bibitem[Mons 2017]{Mons 2017} Barend Mons, Cameron Neylon, Jan Velterop, Michel
Dumontier, Luiz Olavo Bonino da Silva Santos, Mark D. Wilkinson
(2017):\\
\textbf{Cloudy, increasingly FAIR; revisiting the FAIR Data guiding
principles for the European Open Science Cloud}.\\
\emph{Information Services \& Use} \textbf{37}(1)\\
\url{https://doi.org/10.3233/ISU-170824}

\bibitem[Pergl 2019]{Pergl 2019} Robert Pergl, Rob Hooft, Marek Suchánek, Vojtěch
Knaisl, Jan Slifka (2019):\\
\textbf{``Data Stewardship Wizard'': A Tool Bringing Together
Researchers, Data Stewards, and Data Experts around Data Management
Planning}.\\
\emph{Data Science Journal} \textbf{18}(1)\\
\url{https://doi.org/10.5334/dsj-2019-059}

\bibitem[RDF 1.1 2014]{RDF 1.1 2014}~RDF Working Group (2014):\\
\textbf{RDF 1.1 Concepts and Abstract Syntax}.\\
\emph{W3C Recommendation} 25 Feb 2014.
\url{https://www.w3.org/TR/2014/REC-rdf11-concepts-20140225/}.

\bibitem[Schröder 2022]{Schröder 2022} Max Schröder, Susanne Staehlke, Paul Groth, J.
Barbara Nebe, Sascha Spors, Frank Krüger (2022):\\
\textbf{Structure-based knowledge acquisition from electronic lab
notebooks for research data provenance documentation}.\\
\emph{Journal of Biomedical Semantics} \textbf{13}:4\\
\url{https://doi.org/10.1186/s13326-021-00257-x}

\bibitem[Schultes 2019]{Schultes 2019} Erik Schultes, Peter Wittenburg (2019):\\
\textbf{FAIR principles and digital objects: Accelerating convergence on
a data infrastructure}.\\
\emph{Data analytics and management in data intensive domains: 20th
international conference} (DAMDID/RCDL 2018)\\
\url{https://doi.org/10.1007/978-3-030-23584-0_1}

\bibitem[Soiland-Reyes 2022]{Soiland-Reyes 2022} Stian Soiland-Reyes, Peter Sefton, Mercè
Crosas, Leyla Jael Castro, Frederik Coppens, José M. Fernández, Daniel
Garijo, Björn Grüning, Marco La Rosa, Simone Leo, Eoghan Ó Carragáin,
Marc Portier, Ana Trisovic, RO-Crate Community, Paul Groth, Carole Goble
(2022):\\
\textbf{Packaging research artefacts with RO-Crate}.\\
\emph{Data Science} \textbf{5}(2)\\
\url{https://doi.org/10.3233/DS-210053}

\bibitem[Soiland-Reyes 2022a]{Soiland-Reyes 2022a} Stian Soiland-Reyes, Genís Bayarri, Pau
Andrio, Robin Long, Douglas Lowe, Ania Niewielska, Adam Hospital, Paul
Groth (2022):\\
\href{../canonical-workflow-building-blocks/}{\textbf{Making Canonical
Workflow Building Blocks interoperable across workflow languages}}.\\
\emph{Data Intelligence} \textbf{4}(2)\\
\url{https://doi.org/10.1162/dint_a_00135}

\bibitem[Van de Sompel 2022]{Van de Sompel 2022} Herbert Van de Sompel, Martin Klein, Shawn
Jones, Michael L. Nelson, Simeon Warner, Anusuriya Devaraju, Robert
Huber, Wilko Steinhoff, Vyacheslav Tykhonov, Luc Boruta, Enno Meijers,
Stian Soiland-Reyes, Mark Wilkinson (2022):\\
\textbf{FAIR Signposting Profile}. (version 20220727).\\
\url{https://signposting.org/FAIR/}

\bibitem[Walton 2020]{Walton 2020} Stephanie Walton, Laurence Livermore, Olaf Bánki,
Robert Cubey, Robyn Drinkwater, Markus Englund, Carole Goble, Quentin
Groom, Christopher Kermorvant, Isabel Rey, Celia Santos, Ben Scott, Alan
Williams, Zhengzhe Wu (2020):\\
\textbf{Landscape Analysis for the Specimen Data Refinery}.\\
\emph{Research Ideas and Outcomes} \textbf{6}:e57602\\
\url{https://doi.org/10.3897/rio.6.e57602}

\bibitem[Wilkinson 2016]{Wilkinson 2016} Mark D. Wilkinson, Michel Dumontier, IJsbrand Jan
Aalbersberg, Gabrielle Appleton, Myles Axton, Arie Baak, Niklas
Blomberg, Jan-Willem Boiten, Luiz Bonino da Silva Santos, Philip E.
Bourne, Jildau Bouwman, Anthony J. Brookes, Tim Clark, Mercè Crosas,
Ingrid Dillo, Olivier Dumon, Scott Edmunds, Chris T. Evelo, Richard
Finkers, Alejandra Gonzalez-Beltran, Alasdair J.G. Gray, Paul Groth,
Carole Goble, Jeffrey S. Grethe, Jaap Heringa, Peter A.C 't Hoen, Rob
Hooft, Tobias Kuhn, Ruben Kok, Joost Kok, Scott J. Lusher, Maryann E.
Martone, Albert Mons, Abel L. Packer, Bengt Persson, Philippe
Rocca-Serra, Marco Roos, Rene van Schaik, Susanna-Assunta Sansone, Erik
Schultes, Thierry Sengstag, Ted Slater, George Strawn, Morris A. Swertz,
Mark Thompson, Johan van der Lei, Erik van Mulligen, Jan Velterop, Andra
Waagmeester, Peter Wittenburg, Katherine Wolstencroft, Jun Zhao, Barend
Mons (2016):\\
\textbf{The FAIR Guiding Principles for scientific data management and
stewardship}.\\
\emph{Nature Scientific Data} \textbf{3}:160018\\
\url{https://doi.org/10.1038/sdata.2016.18}

%%%%
%%% Below is NOT MERGED
%%% 



%% Chapter 05 references
\section{References from Chapter 5}




\bibitem[Dillen 2019]{Dillen 2019} Mathias Dillen, Quentin Groom, Donat Agosti, Lars Nielsen
(2019):\\
\textbf{Zenodo, an archive and publishing repository: A tale of two
herbarium specimen pilot projects}.\\
\emph{Biodiversity Information Science and Standards} \textbf{3}:e37080
(2019).\\
\url{https://doi.org/10.3897/biss.3.37080}

\bibitem[Droesbeke 2022]{Droesbeke 2022}~Bert Droesbeke, Ignacio Eguinoa, Alban Gaignard, Leo Simone,
Luca Pireddu, Laura Rodríguez-Navas, Stian Soiland-Reyes (2022):\\
\textbf{GitHub -- ResearchObject/ro-crate-py: Python library for
RO-Crate}.\\
\url{https://github.com/researchobject/ro-crate-py}~\\
\url{https://doi.org/10.5281/zenodo.3956493}

\bibitem[Dürst 2005]{Duerst 2005} M. Duerst and M.~Suignard (2005):\\
\textbf{Internationalized resource identifiers (IRIs)}.\\
RFC 3987, \emph{Internet Requests for Comments}, RFC Editor, (2005).\\
\url{https://doi.org/10.17487/rfc3987}

\bibitem[EMBL-EBI 2019]{EMBL-EBI 2019}~EMBL-EBI Microbiome Informatics Team (2019):\\
\textbf{FTP index of /pub/databases/metagenomics/umgs\_analyses/}.\\
\url{http://ftp.ebi.ac.uk/pub/databases/metagenomics/umgs_analyses/}

\bibitem[EMBL-EBI 2020]{EMBL-EBI 2020}~EMBL-EBI Microbiome Informatics Team (2020):\\
\textbf{GitHub -- Finn-Lab/MGS-gut: Analysing Metagenomic Species
(MGS)}.\\
\url{https://github.com/Finn-Lab/MGS-gut}

\bibitem[Ewels 2020]{Ewels 2020}~Philip A Ewels, Alexander Peltzer, Sven Fillinger, Harshil
Patel, Johannes Alneberg, Andreas Wilm, Maxime Ulysse Garcia, Paolo Di
Tommaso, Sven Nahnsen (2020):\\
\textbf{The nf-core framework for community-curated bioinformatics
pipelines}.\\
\emph{Nature Biotechnology} \textbf{38}(3), 276--278.\\
\url{https://doi.org/10.1038/s41587-020-0439-x}

\bibitem[Farnel 2014]{Farnel 2014}~Sharon Farnel, Ali Shiri (2014):\\
\textbf{Metadata for research data: Current practices and trends}.\\
\emph{2014 Proceedings of the International Conference on Dublin Core
and Metadata Applications}, W.~Moen and A.~Rushing, eds, Dublin Core
Metadata Initiative, ISSN 1939-1366.\\
\url{https://dcpapers.dublincore.org/pubs/article/view/3714}.

\bibitem[47]{47}~DONA~Foundation (2018):\\
*\textbf{Digital Object Interface Protocol Specification, version
2.0}.\\
Technical Report.\\
\url{https://www.dona.net/sites/default/files/2018-11/DOIPv2Spec_1.pdf}

\bibitem[48]{48}~Andres Garcia-Silva, Jose Manuel Gomez-Perez, Raul Palma,
Marcin Krystek, Simone Mantovani, Federica Foglini, Valentina Grande,
Francesco De Leo, Stefano Salvi, Elisa Trasatti, Vito Romaniello, Mirko
Albani, Cristiano Silvagni, Rosemarie Leone, Fulvio Marelli, Sergio
Albani, Michele Lazzarini, Hazel J. Napier, Helen M. Glaves, Timothy
Aldridge, Charles Meertens, Fran Boler, Henry W. Loescher, Christine
Laney, Melissa A. Genazzio, Daniel Crawl, Ilkay Altintas (2019):\\
\textbf{Enabling FAIR research in Earth science through research
objects}.\\
\emph{Future Generation Computer Systems} \textbf{98} pp.~550--564.\\
\url{https://arxiv.org/abs/1809.10617}~\\
\url{https://doi.org/10.1016/j.future.2019.03.046}

\bibitem[49]{49}~Peter Sefton, Mike Lynch, Stian Soiland-Reyes (2021):\\
\textbf{GitHub -- UTS-eResearch/ro-crate-js: Research Object Crate
(RO-Crate) utilities}.\\
\url{https://github.com/UTS-eResearch/ro-crate-js}

\bibitem[50]{50}~Ignacio Eguinoa, Stian Soiland-Reyes, Bert Droesbeke, Michael
R. Crusoe (2020):\\
\textbf{GitHub -- workflowhub-eu/galaxy2cwl: Standalone version tool to
get cwl descriptions (initially an abstract cwl interface) of galaxy
workflows and Galaxy workflows executions}.\\
\url{https://github.com/workflowhub-eu/galaxy2cwl}

\bibitem[51]{51}~Marco La Rosa (2021):\\
\textbf{GitHub -- CoEDL/modpdsc},
\url{https://github.com/CoEDL/modpdsc/}

\bibitem[52]{52}~Marco La Rosa (2021):\\
\textbf{GitHub -- CoEDL/ocfl-tools: Tools to process and manipulate an
OCFL tree}.\\
\url{https://github.com/CoEDL/ocfl-tools}

\bibitem[53]{53}~\textbf{Giving software its due}.\\
\emph{Nature Methods} \textbf{16}(3) (2019), 207--207.\\
\url{https://doi.org/10.1038/s41592-019-0350-x}

\bibitem[54]{54}~Carole~Goble (2016):\\
\textbf{What Is Reproducibility? The R* Brouhaha}.\\
\emph{SciRepro Workshop}, TPDL, Hannover, Germany, 2016.
\url{http://repscience2016.research-infrastructures.eu/img/CaroleGoble-ReproScience2016v2.pdf}

\bibitem[55]{55} Carole Goble, Sarah Cohen-Boulakia, Stian Soiland-Reyes, Daniel
Garijo, Yolanda Gil, Michael R. Crusoe, Kristian Peters, Daniel Schober
(2019):\\
\textbf{FAIR Computational Workflows}.\\
\emph{Data Intelligence} \textbf{2}(1--2) pp.~108--121.\\
\url{https://doi.org/10.1162/dint_a_00033}

\bibitem[56]{56} Carole Goble, Stian Soiland-Reyes, Finn Bacall, Stuart Owen,
Alan Williams, Ignacio Eguinoa, Bert Droesbeke, Simone Leo, Luca
Pireddu, Laura Rodriguez-Navas, José Mª Fernández, Salvador
Capella-Gutierrez, Hervé Ménager, Björn Grüning, Beatriz Serrano-Solano,
Philip Ewels, Frederik Coppens (2021):\\
\textbf{Implementing FAIR digital objects in the EOSC-life workflow
collaboratory}.\\
\emph{Zenodo}. \url{https://doi.org/10.5281/zenodo.4605654}

\bibitem[57]{57}~Carole A Goble, Jiten Bhagat, Sergejs Aleksejevs, Don
Cruickshank, Danius Michaelides, David Newman, Mark Borkum, Sean
Bechhofer, Marco Roos, Peter Li, David De Roure (2010):\\
\textbf{myExperiment: A repository and social network for the sharing of
bioinformatics workflows}.\\
\emph{Nucleic Acids Research} \textbf{38}(Web Server issue)
W677--W682.\\
\url{https://doi.org/10.1093/nar/gkq429}

\bibitem[58]{58}~Alasdair Gray, Carole Goble, Rafael Jimenez, Bioschemas
Community(2017):\\
\textbf{Bioschemas: From Potato Salad to Protein Annotation}.\\
\emph{ISWC}, Vienna, Austria.\\
\url{https://iswc2017.semanticweb.org/paper-579/}

\bibitem[59]{59}~Robert L Grossman, Allison Heath, Mark Murphy, Maria Patterson,
Walt Wells (2016):\\
\textbf{A case for data commons: Toward data science as a service}.\\
\emph{Computing in Science \& Engineering} \textbf{18}(5) pp.~10--20.\\
\url{https://doi.org/10.1109/MCSE.2016.92}

\bibitem[60]{60}~Björn Grüning, John Chilton, Johannes Köster, Ryan Dale, Nicola
Soranzo, Marius van den Beek, Jeremy Goecks, Rolf Backofen, Anton
Nekrutenko, James Taylor (2018):\\
\textbf{Practical computational reproducibility in the life sciences}.\\
\emph{Cell Systems} \textbf{6}(6) pp.~631--635.\\
\url{https://doi.org/10.1016/j.cels.2018.03.014}

\bibitem[61]{61}~Björn Grüning, Ryan Dale, Andreas Sjödin, Brad A Chapman,
Jillian Rowe, Christopher H Tomkins-Tinch, Renan Valieris, Johannes
Köster, Bioconda Team (2018):\\
\textbf{Bioconda: Sustainable and comprehensive software distribution
for the life sciences}.\\
\emph{Nature Methods} \textbf{15}(7) pp.~475--476.\\
\url{https://doi.org/10.1038/s41592-018-0046-7}

\bibitem[62]{62} Ramanathan V Guha, Dan Brickley, Steve Macbeth (2015):\\
\textbf{Schema.org: Evolution of Structured Data on the Web: Big data
makes common schemas even more necessary}.\\
\emph{Queue} \textbf{13}(9) pp.~10--37.\\
\url{https://doi.org/10.1145/2857274.2857276}

\bibitem[63]{63}~Tom Heath, Christian Bizer (2011):\\
\textbf{Linked Data: Evolving the Web into a Global Data Space}.\\
\emph{Synthesis Lectures on the Semantic Web: Theory and Technology}
\textbf{1} pp.~1--136, ISSN 2160-4711. ISBN 9781608454310 / ISBN
9781608454303. \url{https://identifiers.org/isbn/9781608454303}\\
\url{https://doi.org/10.2200/S00334ED1V01Y201102WBE001}

\bibitem[64]{64}~\textbf{IEEE Standard for Bioinformatics Analyses Generated by
High-Throughput Sequencing (HTS) to Facilitate Communication} (2020).\\
\emph{IEEE Std} \textbf{2791-2020}.\\
ISBN 978-1-5044-6466-6.\\
\url{https://www.research.manchester.ac.uk/portal/en/publications/ieee-2791(936de52b-ac53-4f0e-9927-77fd7073e88d).html}~\\
\url{https://doi.org/10.1109/ieeestd.2020.9094416}

\bibitem[65]{65}~Mark A Jensen, Vincent Ferretti, Robert L Grossman, Louis M
Staudt (2017):\\
\textbf{The NCI Genomic Data Commons as an engine for precision
medicine}.\\
\emph{Blood} \textbf{130}(4) pp.~453--459.\\
\url{https://doi.org/10.1182/blood-2017-03-735654}

\bibitem[66]{66}~Matthew B. Jones, Stephen Richard, Dave Vieglais, Adam
Shepherd, Ruth Duerr, Dougl Fils, Lewis McGibbney (2021):\\
\textbf{Science-on-Schema.org v1.2.0}\\
\url{https://doi.org/10.5281/zenodo.4477164}

\bibitem[67]{67} Megan Katsumi, Michael Grüninger (2016):\\
\textbf{What is ontology reuse?}.\\
In: \emph{Formal Ontology in Information Systems}, R.~Ferrario and
W.~Kuhn, eds,\\
\emph{Frontiers in Artificial Intelligence and Applications}
\textbf{283}\\
ISBN 978-1-61499-660-6.\\
\url{https://doi.org/10.3233/978-1-61499-660-6-9}

\bibitem[68]{68}~Farah Zaib Khan, Stian Soiland-Reyes, Richard O. Sinnott,
Andrew Lonie, Carole Goble, Michael R. Crusoe (2019):\\
\textbf{Sharing interoperable workflow provenance: A review of best
practices and their practical application in CWLProv}.\\
\emph{GigaScience} \textbf{8}(11).\\
\url{https://doi.org/10.1093/gigascience/giz095}

\bibitem[69]{69}~Jihie Kim, Ewa Deelman, Yolanda Gil, Gaurang Mehta, Varun
Ratnakar (2008):\\
\textbf{Provenance trails in the Wings/Pegasus system}.\\
\emph{Concurrency and Computation: Practice and Experience}
\textbf{20}(5) pp.~587--597.\\
\url{https://doi.org/10.1002/cpe.1228}

\bibitem[70]{70}~Thomas Kluyver, Benjamin Ragan-Kelley, Fernando Pérez, Brian
Granger, Matthias Bussonnier, Jonathan Frederic, Kyle Kelley, Jessica
Hamrick, Jason Grout, Sylvain Corlay, Paul Ivanov, Damián Avila, Safia
Abdalla, Carol Willing, Jupyter Development Team (2016):\\
\textbf{Jupyter Notebooks -- a publishing format for reproducible
computational workflows}.\\
in: \emph{Positioning and Power in Academic Publishing: Players, Agents
and Agendas},\\
\emph{Proceedings of the 20th International Conference on Electronic
Publishing}, pp.~87--90, ISBN 978-1-61499-649-1.\\
\url{https://doi.org/10.3233/978-1-61499-649-1-87}

\bibitem[71]{71}~Laura Koesten, Kathleen Gregory, Paul Groth, Elena Simperl
(2021):\\
\textbf{Talking datasets -- understanding data sensemaking
behaviours}.\\
\emph{International journal of human-computer studies}
\textbf{146}:102562.\\
\url{https://doi.org/10.1016/j.ijhcs.2020.102562}

\bibitem[72]{72}~Laura Koesten, Pavlos Vougiouklis, Elena Simperl, Paul Groth
(2020):\\
\textbf{Dataset reuse: Toward translating principles to practice}.\\
\emph{Patterns} \textbf{1}(8):100136.\\
\url{https://doi.org/10.1016/j.patter.2020.100136}

\bibitem[73]{73}~Johannes Köster, Sven Rahmann (2012):\\
\textbf{Snakemake -- a scalable bioinformatics workflow engine}.\\
\emph{Bioinformatics} \textbf{28}(19) pp.~2520--2522.\\
\url{https://doi.org/10.1093/bioinformatics/bts480}

\bibitem[74]{74}~J.~Kunze, J.~Littman, E.~Madden, J.~Scancella, C.~Adams
(2018):\\
\textbf{The BagIt File Packaging Format}, (V1.0), RFC 8493,
\emph{Internet Requests for Comments}, RFC Editor.\\
\url{https://doi.org/10.17487/RFC8493}

\bibitem[75]{75}~Krzysztof Kurowski, Oscar Corcho, Christine Choirat, Magnus
Eriksson, Frederik Coppens, Mark van de Sanden, Milan Ojsteršek
(2021):\\
\textbf{EOSC Interoperability Framework}.\\
\emph{Publications Office of the EU}, Technical Report, 2021.\\
\url{https://doi.org/10.2777/620649}

\bibitem[76]{76}~Kyle Chard, Niall Gaffney, Mihael Hategan, Kacper Kowalik,
Bertram Ludäscher, Timothy McPhillips, Jarek Nabrzyski, Victoria
Stodden, Ian Taylor, Thomas Thelen, Matthew J. Turk, Craig Willis
(2020):\\
\textbf{Toward enabling reproducibility for data-intensive research
using the Whole Tale platform}.\\
\emph{Advances in Parallel Computing} \textbf{36} pp 766--778.\\
\url{https://doi.org/10.3233/APC200107}

\bibitem[77]{77}~M.~La Rosa (2021):\\
\textbf{Arkisto Platform: Describo Online}.\\
\url{https://arkisto-platform.github.io/describo-online/}

\bibitem[78]{78}~M.~La Rosa and Peter Sefton (2021):\\
\textbf{Arkisto Platform: Describo}.\\
\url{https://arkisto-platform.github.io/describo/}

\bibitem[79]{79}~R.~Lammey (2020):\\
\textbf{Solutions for identification problems: A look at the research
organization registry}.\\
\emph{Science Editing} \textbf{7}(1) pp.~65--69.\\
\url{https://doi.org/10.6087/kcse.192}

\bibitem[80]{80}~Anna-Lena Lamprecht, Leyla Garcia, Mateusz Kuzak, Carlos
Martinez, Ricardo Arcila, Eva Martin Del Pico, Victoria Dominguez Del
Angel, Stephanie Van De Sandt, Jon Ison, Paula Andrea Martinez, Peter
Mcquilton, Alfonso Valencia, Jennifer Harrow, Fotis Psomopoulos, Josep
Ll. Gelpi, Neil Chue Hong, Carole Goble, Salvador Capella-Gutierrez
(2019):\\
\textbf{Towards FAIR principles for research software}.\\
\emph{Data Science} \textbf{3}(1) pp.~1--23.\\
\url{https://doi.org/10.3233/DS-190026}

\bibitem[81]{81}~T.~Lebo, S.~Sahoo, D.~McGuinness, K.~Belhajjame, J.~Cheney,
D.~Corsar, D.~Garijo, Stian~Soiland-Reyes, S.~Zednik and J.~Zhao
(2013):\\
\textbf{PROV-O: The PROV Ontology}.\\
\emph{W3C Recommendation} 30 April 2013.
\url{https://www.w3.org/TR/2013/REC-prov-o-20130430/}

\bibitem[82]{82}~J.~Leipzig, D.~Nüst, C.T.~Hoyt, K.~Ram and J.~Greenberg
(2021):\\
\textbf{The role of metadata in reproducible computational research}.\\
\emph{Patterns} \textbf{2}(9):100322.\\
\url{https://doi.org/10.1016/j.patter.2021.100322}

\bibitem[83]{83}~D.~Lowe and G.~Bayarri (2021):\\
\textbf{Protein Ligand Complex MD Setup tutorial using BioExcel Building
Blocks (biobb) (jupyter notebook)}.\\
\url{https://doi.org/10.48546/workflowhub.workflow.56.1}

\bibitem[84]{84}~M.~Lynch and Peter Sefton (2022):\\
\textbf{npm: ro-crate-excel}.\\
\emph{npm} \url{https://www.npmjs.com/package/ro-crate-excel}

\bibitem[85]{85}~GitHub (2021):\\
\textbf{Managing large files -- GitHub Docs}.\\
\url{https://docs.github.com/en/repositories/working-with-files/managing-large-files}

\bibitem[86]{86}~Julie A McMurry, Nick Juty, Niklas Blomberg, Tony Burdett, Tom
Conlin, Nathalie Conte, Mélanie Courtot, John Deck, Michel Dumontier,
Donal K Fellows, Alejandra Gonzalez-Beltran, Philipp Gormanns, Jeffrey
Grethe, Janna Hastings, Jean-Karim Hériché, Henning Hermjakob, Jon C
Ison, Rafael C Jimenez, Simon Jupp, John Kunze, Camille Laibe, Nicolas
Le Novère, James Malone, Maria Jesus Martin, Johanna R McEntyre, Chris
Morris, Juha Muilu, Wolfgang Müller, Philippe Rocca-Serra,
Susanna-Assunta Sansone, Murat Sariyar, Jacky L Snoep, Stian
Soiland-Reyes, Natalie J Stanford, Neil Swainston, Nicole Washington,
Alan R Williams, Sarala M Wimalaratne, Lilly M Winfree, Katherine
Wolstencroft, Carole Goble, Cristopher J Mungall, Melissa A Haendel,
Helen Parkinson (2017):\\
\textbf{Identifiers for the 21st century: How to design, provision, and
reuse persistent identifiers to maximize utility and impact of life
science data}.\\
\emph{PLOS Biology} \textbf{15}(6):e2001414.\\
\url{https://doi.org/10.1371/journal.pbio.2001414}

\bibitem[87]{87}~T.~Miksa, M.~Jaoua and G.~Arfaoui (2020):\\
\textbf{Research object crates and machine-actionable data management
plans}.\\
\emph{1st Workshop on Research Data Management for Linked Open
Science}.\\
\url{https://doi.org/10.4126/frl01-006423291}

\bibitem[88]{88}~T.~Miksa, S.~Simms, D.~Mietchen and S.~Jones (2019):\\
\textbf{Ten principles for machine-actionable data management plans}.\\
\emph{PLOS Computational Biology} \textbf{15}(3): e1006750.\\
\url{https://doi.org/10.1371/journal.pcbi.1006750}

\bibitem[89]{89}~Steffen Möller, Hajo Nils Krabbenhöft, Andreas Tille, David
Paleino, Alan Williams, Katy Wolstencroft, Carole Goble, Richard
Holland, Dominique Belhachemi, Charles Plessy (2010):\\
\textbf{Community-driven computational biology with Debian Linux}.\\
\emph{BMC Bioinformatics} \textbf{11}(Suppl 12):S5.\\
\url{https://doi.org/10.1186/1471-2105-11-S12-S5}

\bibitem[90]{90}~Steffen Möller, Stuart W. Prescott, Lars Wirzenius; Petter
Reinholdtsen, Brad Chapman, Pjotr Prins, Stian Soiland-Reyes, Fabian
Klötzl, Andrea Bagnacani, Matúš Kalaš, Andreas Tille, Michael R. Crusoe
(2017):\\
\textbf{Robust cross-platform workflows: How technical and scientific
communities collaborate to develop, test and share best practices for
data analysis}.\\
\emph{Data Science and Engineering} \textbf{2}(3) pp.~232--244.\\
\url{https://doi.org/10.1007/s41019-017-0050-4}

\bibitem[91]{91}~Barend~Mons (2018):\\
\textbf{Data Stewardship for Open Science}, 1st edn. Taylor \& Francis,
p.~240. \href{https://identifiers.org/isbn/9781315351148}{ISBN
9781315351148}.

\bibitem[92]{92}~myExperiment (2009):\\
\textbf{myExperiment Ontology Modules}.\\
\emph{myExperiment} / \emph{Internet Archive}\\
\url{https://web.archive.org/web/20091115080336/http\%3a\%2f\%2frdf.myexperiment.org/ontologies}

\bibitem[93]{93}~D.~Newman, S.~Bechhofer and D.~De Roure (2009):\\
\textbf{myExperiment: An ontology for e-Research}.\\
in: \emph{Proceedings of the Workshop on Semantic Web Applications in
Scientific Discourse (SWASD 2009)}, T.~Clark, J.S.~Luciano,
M.S.~Marshall, E.~Prud'Hommeaux and S.~Stephens, eds,\\
\emph{CEUR Workshop Proceedings} \textbf{523}. ISSN 1613-0073.\\
\url{http://ceur-ws.org/Vol-523/Newman.pdf}

\bibitem[94]{94}~Cameron~Neylon (2017):\\
\textbf{As a researcher \ldots{} I'm a bit bloody fed up with Data
Management}.\\
\emph{Science in the Open} (blog)
\url{https://cameronneylon.net/blog/as-a-researcher-im-a-bit-bloody-fed-up-with-data-management/}.

\bibitem[95]{95}~npm:\\
\textbf{ro-crate-html-js}\\
\url{https://www.npmjs.com/package/ro-crate-html-js}

\bibitem[96]{96}~\textbf{OCFL, Oxford Common File Layout Specification},
Recommendation, 2020.\\
\url{https://ocfl.io/1.0/spec/}

\bibitem[97]{97}~A.~Piper (2020):\\
\textbf{Digital crowdsourcing and public understandings of the past:
Citizen historians meet criminal characters}.\\
\emph{History Australia} \textbf{17}(3) pp.~525--541.\\
\url{https://doi.org/10.1080/14490854.2020.1796500}


\bibitem[99]{99}~Heidi L. Rehm, Angela J.H. Page, Lindsay Smith, Jeremy B.
Adams, Gil Alterovitz, Lawrence J. Babb, Maxmillian P. Barkley, Michael
Baudis, Michael J.S. Beauvais, Tim Beck, Jacques S. Beckmann, Sergi
Beltran, David Bernick, Alexander Bernier, James K. Bonfield, Tiffany F.
Boughtwood, Guillaume Bourque, Sarion R. Bowers, Anthony J. Brookes,
Michael Brudno, Matthew H. Brush, David Bujold, Tony Burdett, Orion J.
Buske, Moran N. Cabili, Daniel L. Cameron, Robert J. Carroll, Esmeralda
Casas-Silva, Debyani Chakravarty, Bimal P. Chaudhari, Shu Hui Chen, J.
Michael Cherry, Justina Chung, Melissa Cline, Hayley L. Clissold, Robert
M. Cook-Deegan, Mélanie Courtot, Fiona Cunningham, Miro Cupak, Robert M.
Davies, Danielle Denisko, Megan J. Doerr, Lena I. Dolman, Edward S.
Dove, L. Jonathan Dursi, Stephanie O.M. Dyke, James A. Eddy, Karen
Eilbeck, Kyle P. Ellrott, Susan Fairley, Khalid A. Fakhro, Helen V.
Firth, Michael S. Fitzsimons, Marc Fiume, Paul Flicek, Ian M. Fore,
Mallory A. Freeberg, Robert R. Freimuth, Lauren A. Fromont, Jonathan
Fuerth, Clara L. Gaff, Weiniu Gan, Elena M. Ghanaim, David Glazer,
Robert C. Green, Malachi Griffith, Obi L. Griffith, Robert L. Grossman,
Tudor Groza, Jaime M. Guidry Auvil, Roderic Guigó, Dipayan Gupta,
Melissa A. Haendel, Ada Hamosh, David P. Hansen, Reece K. Hart, Dean
Mitchell Hartley, David Haussler, Rachele M. Hendricks-Sturrup, Calvin
W.L. Ho, Ashley E. Hobb, Michael M. Hoffman, Oliver M. Hofmann, Petr
Holub, Jacob Shujui Hsu, Jean-Pierre Hubaux, Sarah E. Hunt, Ammar
Husami, Julius O. Jacobsen, Saumya S. Jamuar, Elizabeth L. Janes,
Francis Jeanson, Aina Jené, Amber L. Johns, Yann Joly, Steven J.M.
Jones, Alexander Kanitz, Kazuto Kato, Thomas M. Keane, Kristina
Kekesi-Lafrance, Jerome Kelleher, Giselle Kerry, Seik-Soon Khor, Bartha
M. Knoppers, Melissa A. Konopko, Kenjiro Kosaki, Martin Kuba, Jonathan
Lawson, Rasko Leinonen, Stephanie Li, Michael F. Lin, Mikael Linden,
Xianglin Liu, Isuru Udara Liyanage, Javier Lopez, Anneke M. Lucassen,
Michael Lukowski, Alice L. Mann, John Marshall, Michele Mattioni,
Alejandro Metke-Jimenez, Anna Middleton, Richard J. Milne, Fruzsina
Molnár-Gábor, Nicola Mulder, Monica C. Munoz-Torres, Rishi Nag, Hidewaki
Nakagawa, Jamal Nasir, Arcadi Navarro, Tristan H. Nelson, Ania
Niewielska, Amy Nisselle, Jeffrey Niu, Tommi H. Nyrönen, Brian D.
O'Connor, Sabine Oesterle, Soichi Ogishima, Vivian Ota Wang, Laura A.D.
Paglione, Emilio Palumbo, Helen E. Parkinson, Anthony A. Philippakis,
Angel D. Pizarro, Andreas Prlic, Jordi Rambla, Augusto Rendon, Renee A.
Rider, Peter N. Robinson, Kurt W. Rodarmer, Laura Lyman Rodriguez, Alan
F. Rubin, Manuel Rueda, Gregory A. Rushton, Rosalyn S. Ryan, Gary I.
Saunders, Helen Schuilenburg, Torsten Schwede, Serena Scollen, Alexander
Senf, Nathan C. Sheffield, Neerjah Skantharajah, Albert V. Smith, Heidi
J. Sofia, Dylan Spalding, Amanda B. Spurdle, Zornitza Stark, Lincoln D.
Stein, Makoto Suematsu, Patrick Tan, Jonathan A. Tedds, Alastair A.
Thomson, Adrian Thorogood, Timothy L. Tickle, Katsushi Tokunaga, Juha
Törnroos, David Torrents, Sean Upchurch, Alfonso Valencia, Roman Valls
Guimera, Jessica Vamathevan, Susheel Varma, Danya F. Vears, Coby Viner,
Craig Voisin, Alex H. Wagner, Susan E. Wallace, Brian P. Walsh, Marc S.
Williams, Eva C. Winkler, Barbara J. Wold, Grant M. Wood, J. Patrick
Woolley, Chisato Yamasaki, Andrew D. Yates, Christina K. Yung, Lyndon J.
Zass, Ksenia Zaytseva, Junjun Zhang, Peter Goodhand, Kathryn North, Ewan
Birney (2021):\\
\textbf{GA4GH: International policies and standards for data sharing
across genomic research and healthcare}.\\
\emph{Cell Genomics} \textbf{1}(2):100029.\\
\url{https://doi.org/10.1016/j.xgen.2021.100029}

\bibitem[100]{100}~N.~Rettberg and B.~Schmidt (2015):\\
\textbf{OpenAIRE: Supporting a European open access mandate}.\\
\emph{College \& Research Libraries News} \textbf{76}(6) pp.~306--310.
\url{http://resolver.sub.uni-goettingen.de/purl?gs-1/11942}\\
\url{https://doi.org/10.5860/crln.76.6.9326}

\bibitem[101]{101}~G.K.~Sandve, A.~Nekrutenko, J.~Taylor and E.~Hovig (2013):\\
\textbf{Ten simple rules for reproducible computational research}.\\
\emph{PLOS Computational Biology} \textbf{9}(10):e1003285.\\
\url{https://doi.org/10.1371/journal.pcbi.1003285}

\bibitem[102]{102} Lynn M. Schriml, Maria Chuvochina, Neil Davies, Emiley A.
Eloe-Fadrosh, Robert D. Finn, Philip Hugenholtz, Christopher I. Hunter,
Bonnie L. Hurwitz, Nikos C. Kyrpides, Folker Meyer, Ilene Karsch
Mizrachi, Susanna-Assunta Sansone, Granger Sutton, Scott Tighe, Ramona
Walls (2020):\\
\textbf{COVID-19 pandemic reveals the peril of ignoring metadata
standards}.\\
\emph{Scientific Data} \textbf{7}(1):188.\\
\url{https://doi.org/10.1038/s41597-020-0524-5}

\bibitem[103]{103}~Peter Sefton, G.~Devine, C.~Evenhuis, M.~Lynch, S.~Wise,
M.~Lake and D.~Loxton (2018):\\
\textbf{DataCrate: a method of packaging, distributing, displaying and
archiving Research Objects}.\\
in: \emph{Workshop on Research Objects (RO 2018)}, 29 Oct 2018 at IEEE
eScience 2018, Amsterdam, Netherland. \emph{Zenodo}\\
\url{https://doi.org/10.5281/zenodo.1445817}

\bibitem[Sefton 2021]{Sefton 2021}~Peter Sefton (2021):\\
\textbf{FAIR Data Management; It's a lifestyle not a lifecycle}.\\
\emph{ptsefton.com}. \url{http://ptsefton.com/2021/04/07/rdmpic/}

\bibitem[105]{105}~Peter Sefton, Eoghan Ó Carragáin, Stian Soiland-Reyes, Oscar
Corcho, Daniel Garijo, Raul Palma, Frederik Coppens, Carole Goble, José
María Fernández, Kyle Chard, Jose Manuel Gomez-Perez, Michael R Crusoe,
Ignacio Eguinoa, Nick Juty, Kristi Holmes, Jason A. Clark, Salvador
Capella-Gutierrez, Alasdair J. G. Gray, Stuart Owen, Alan R. Williams,
Giacomo Tartari, Finn Bacall, Thomas Thelen (2019):\\
\textbf{RO-Crate Metadata Specification 1.0}.\\
\url{https://doi.org/10.5281/zenodo.3541888}

\bibitem[106]{106} Peter Sefton, Eoghan Ó Carragáin, Stian Soiland-Reyes, Oscar
Corcho, Daniel Garijo, Raul Palma, Frederik Coppens, Carole Goble, José
María Fernández, Kyle Chard, Jose Manuel Gomez-Perez, Michael R Crusoe,
Ignacio Eguinoa, Nick Juty, Kristi Holmes, Jason A. Clark, Salvador
Capella-Gutierrez, Alasdair J. G. Gray, Stuart Owen, Alan R. Williams,
Giacomo Tartari, Finn Bacall, Thomas Thelen, Hervé Ménager, Laura
Rodríguez-Navas, Paul Walk, brandon whitehead, Mark Wilkinson, Paul
Groth, Erich Bremer, LJ Garcia Castro, Karl Sebby, Alexander Kanitz, Ana
Trisovic, Gavin Kennedy, Mark Graves, Jasper Koehorst, Simone Leo, Marc
Portier (2021):\\
\textbf{RO-Crate Metadata Specification 1.1.1}.\\
\url{https://doi.org/10.5281/zenodo.4541002}

\bibitem[107]{107}~Peter Sefton, Eoghan Ó Carragáin, Stian Soiland-Reyes, Oscar
Corcho, Daniel Garijo, Raul Palma, Frederik Coppens, Carole Goble, José
María Fernández, Kyle Chard, Jose Manuel Gomez-Perez, Michael R Crusoe,
Ignacio Eguinoa, Nick Juty, Kristi Holmes, Jason A. Clark, Salvador
Capella-Gutierrez, Alasdair J. G. Gray, Stuart Owen, Alan R. Williams,
Giacomo Tartari, Finn Bacall, Thomas Thelen, Hervé Ménager, Laura
Rodríguez-Navas, Paul Walk, brandon whitehead, Mark Wilkinson, Paul
Groth, Erich Bremer, LJ Garcia Castro, Karl Sebby, Alexander Kanitz, Ana
Trisovic, Gavin Kennedy, Mark Graves, Jasper Koehorst, Simone Leo
(2020):\\
\textbf{RO-Crate Metadata Specification 1.1}.\\
\url{https://doi.org/10.5281/zenodo.4031327}

\bibitem[108]{108}~Stian~Soiland-Reyes (2020):\\
\textbf{I am looking for which bioinformatics journals encourage authors
to submit their code/pipeline/workflow supporting data analysis},
\emph{Twitter}
\url{https://twitter.com/soilandreyes/status/1250721245622079488}

\bibitem[109]{109}~Stian~Soiland-Reyes (2021):\\
\textbf{Describing and packaging workflows using RO-Crate and BioCompute
Objects} \emph{Zenodo}, Webinar for U.S. Food and Drug Administration
(FDA), 2021-05-12.\\
\url{https://doi.org/10.5281/zenodo.4633732}

\bibitem[110]{110}~Stian~Soiland-Reyes, P.~Alper and Carole~Goble (2016):\\
\textbf{Tracking Workflow Execution With TavernaPROV},
\emph{ProvenanceWeek 2016}, session ``PROV: Three Years Later''.
\url{https://s11.no/2016/provweek-tavernaprov/}\\
\url{https://doi.org/10.5281/zenodo.51314}

\bibitem[111]{111}~Stian~Soiland-Reyes, M.~Gamble and R.~Haines (2014):\\
\textbf{Research Object Bundle 1.0}.\\
\url{https://w3id.org/bundle/2014-11-05/}
\url{https://doi.org/10.5281/zenodo.12586}.

\bibitem[112]{112}~M.~Sporny, D.~Longley, G.~Kellogg, M.~Lanthaler and
N.~Lindström (2014):\\
\textbf{JSON-LD 1.0}, W3C Recommendation.
\url{https://www.w3.org/TR/2014/REC-json-ld-20140116/}

%\bibitem[113]{113}~V.~Stodden, M.~McNutt, D.H.~Bailey, E.~Deelman, Y.~Gil,
%B.~Hanson, M.A.~Heroux, J.P.A.~Ioannidis and M.~Taufer (2016):\\
%\textbf{Enhancing reproducibility for computational methods}.\\
%\emph{Science} \textbf{354}(6317) pp.~1240--1241.\\
%\url{https://doi.org/10.1126/science.aah6168}

\bibitem[114]{114}~N.~Thieberger and L.~Barwick (2012):\\
\textbf{Keeping records of language diversity in melanesia: The Pacific
and regional archive for digital sources in endangered cultures
(PARADISEC)}, in: \emph{Melanesian Languages on the Edge of Asia:
Challenges for the 21st Century}, N.~Evans and M.~Klamer, eds,\\
\emph{Language Documentation \& Conservation Special Publication}
\textbf{SP05} University of Hawai'i Press, pp.~239--253. ISBN
978-0-9856211-2-4\\
\url{http://hdl.handle.net/10125/4567}

\bibitem[115]{115}~\textbf{Tools: Data Portal \& Discovery}.\\
\url{https://arkisto-platform.github.io/tools/portal/}

\bibitem[116]{116}~R.~Troncy, W.~Bailer, M.~Höffernig and M.~Hausenblas (2010):\\
\textbf{VAMP: A service for validating MPEG-7 descriptions w.r.t. to
formal profile definitions}.\\
\emph{Multimedia tools and applications} \textbf{46}(2--3)
pp.~307--329.\\
\url{https://www.persistent-identifier.nl/urn:nbn:nl:ui:18-14511}~\\
\url{https://doi.org/10.1007/s11042-009-0397-2}

\bibitem[117]{117}~Herbert~Van de Sompel, Carl~Lagoze (2007):\\
\textbf{Interoperability for the discovery, use, and re-use of units of
scholarly communication}.\\
\emph{CTWatch Quarterly} \textbf{3}(3).\\
\url{http://icl.utk.edu/ctwatch/quarterly/articles/2007/08/interoperability-for-the-discovery-use-and-re-use-of-units-of-scholarly-communication/}

\bibitem[118]{118}~T.~Vergoulis, K.~Zagganas, L.~Kavouras, M.~Reczko,
S.~Sartzetakis and T.~Dalamagas (2021):\\
\textbf{SCHeMa: Scheduling Scientific Containers on a Cluster of
Heterogeneous Machines}.\\
\url{https://arxiv.org/abs/2103.13138v1}

\bibitem[119]{119}~C.J.~Volk, Y.~Lucero and K.~Barnas (2014):\\
\textbf{Why is data sharing in collaborative natural resource efforts so
hard and what can we do to improve it?}.\\
\emph{Environmental Management} \textbf{53}(5) pp.~883--893.\\
\url{https://doi.org/10.1007/s00267-014-0258-2}

\bibitem[120]{120}~W3C Technical Architecture Group (2007):\\
\textbf{Dereferencing HTTP URIs}.\\
Draft Tag Finding, 2007.
\url{https://www.w3.org/2001/tag/doc/httpRange-14/2007-08-31/HttpRange-14.html}

\bibitem[121]{121}~P.~Walk, T.~Miksa and P.~Neish (2019):\\
\textbf{RDA DMP Common Standard for Machine-Actionable Data Management
Plans}.\\
\emph{Research Data Alliance}\\
\url{https://doi.org/10.15497/rda00039}

\bibitem[122]{122}~Stephanie Walton, Laurence Livermore, Olaf Bánki, Robert W. N.
Cubey, Robyn Drinkwater, Markus Englund, Carole Goble, Quentin Groom,
Christopher Kermorvant, Isabel Rey, Celia M Santos, Ben Scott, Alan R.
Williams, Zhengzhe Wu (2020):\\
\textbf{Landscape analysis for the specimen data refinery}.\\
\emph{Research Ideas and Outcomes} \textbf{6}.\\
\url{https://doi.org/10.3897/rio.6.e57602}

\bibitem[123]{123}~Mark D. Wilkinson, Michel Dumontier, IJsbrand Jan Aalbersberg,
Gabrielle Appleton, Myles Axton, Arie Baak, Niklas Blomberg, Jan-Willem
Boiten, Luiz Bonino da Silva Santos, Philip E. Bourne, Jildau Bouwman,
Anthony J. Brookes, Tim Clark, Mercè Crosas, Ingrid Dillo, Olivier
Dumon, Scott Edmunds, Chris T. Evelo, Richard Finkers, Alejandra
Gonzalez-Beltran, Alasdair J.G. Gray, Paul Groth, Carole Goble, Jeffrey
S. Grethe, Jaap Heringa, Peter A.C 't Hoen, Rob Hooft, Tobias Kuhn,
Ruben Kok, Joost Kok, Scott J. Lusher, Maryann E. Martone, Albert Mons,
Abel L. Packer, Bengt Persson, Philippe Rocca-Serra, Marco Roos, Rene
van Schaik, Susanna-Assunta Sansone, Erik Schultes, Thierry Sengstag,
Ted Slater, George Strawn, Morris A. Swertz, Mark Thompson, Johan van
der Lei, Erik van Mulligen, Jan Velterop, Andra Waagmeester, Peter
Wittenburg, Katherine Wolstencroft, Jun Zhao, Barend Mons (2016):\\
\textbf{The FAIR guiding principles for scientific data management and
stewardship}.\\
\emph{Scientific Data} \textbf{3}:160018.\\
\url{https://doi.org/10.1038/sdata.2016.18}

\bibitem[124]{124}~\textbf{WorkflowHub project: Project pages for developing and
running the WorkflowHub, a registry of scientific workflows}.\\
\url{https://w3id.org/workflowhub/}

\bibitem[125]{125}~Jun Zhao, Jose Manuel Gomez-Perezy, Khalid Belhajjame, Graham
Klyne, Esteban Garcia-Cuestay, Aleix Garridoy, Kristina Hettne, Marco
Roos, David De Roure, Carole Goble (2012):\\
\textbf{Why workflows break -- understanding and combating decay in
taverna workflows}.\\
\emph{2012 IEEE 8th International Conference on e-Science}, IEEE.
\url{https://www.research.manchester.ac.uk/portal/files/174861334/why_decay.pdf}
ISBN 978-1-4673-4466-1.\\
\url{https://doi.org/10.1109/eScience.2012.6404482}

\bibitem[126]{126}~F.~Zoubek and M.~Winkler (2021):\\
\textbf{RO Crates and Excel}.\\
\url{https://github.com/e11938258/RO-Crates-and-Excel}
\url{https://doi.org/10.5281/zenodo.5068950}

\bibitem[127]{127}~M.~Žumer (2009):\\
\textbf{National Bibliographies in the Digital Age: Guidance and New
Directions}.\\
\emph{IFLA Series on Bibliographic Control}, IFLA Working Group on
Guidelines for National Bibliographies, Walter de Gruyter -- K. G. Saur,
2009, ISSN 1868-8438. ISBN 9783598441844.\\
\url{https://doi.org/10.1515/9783598441844}


%% Chapter 06 references
\section{References from Chapter 6}

\bibitem[Stodden 2016]{1} Victoria Stodden, Marcia McNutt, David H. Bailey, Ewa Deelman,
Yolanda Gil, Brooks Hanson, Michael A. Heroux, John P.A. Ioannidis,
Michela Taufer (2016):\\
\textbf{Enhancing reproducibility for computational methods.}
\emph{Science} \textbf{354}(6317) pp 1240--1241\\
\url{https://doi.org/10.1126/science.aah6168}

\bibitem[2]{2} Jeremy Leipzig, Daniel Nüst, Charles Tapley Hoyt, Karthik Ram,
Jane Greenberg (2021):\\
\textbf{The role of metadata in reproducible computational research}.
\emph{Patterns} \textbf{2}(9):100322.\\
\url{https://doi.org/10.1016/j.patter.2021.100322}

\bibitem[3]{3} Daniel S. Katz, Morane Gruenpeter, Tom Honeyman, Lorraine Hwang,
Mark D. Wilkinson, Vanessa Sochat, Hartwig Anzt, Carole Goble, FAIR4RS
Subgroup 1 (2021):\\
\textbf{A Fresh Look at FAIR for Research Software}.\\
\href{https://arxiv.org/abs/2101.10883}{arXiv:2101.10883}

\bibitem[4]{4} Steffen Möller, Stuart W. Prescott, Lars Wirzenius, Petter
Reinholdtsen, Brad Chapman, Pjotr Prins, Stian Soiland-Reyes, Fabian
Klötzl, Andrea Bagnacani, Matúš Kalaš, Andreas Tille, Michael R. Crusoe
(2017):\\
\textbf{Robust cross-platform workflows: How technical and scientific
communities collaborate to develop, test and share best practices for
data analysis}. \emph{Data Science and Engineering} \textbf{2} pp
232--244.\\
\url{https://doi.org/10.1007/s41019-017-0050-4}

\bibitem[5]{5} Sarah Cohen-Boulakia, Khalid Belhajjame, Olivier Collin, Jérôme
Chopard, Christine Froidevaux, Alban Gaignard, Konrad Hinsen, Pierre
Larmande, Yvan Le Bras, Frédéric Lemoine, Fabien Mareuil, Hervé Ménager,
Christophe Pradal, Christophe Blanchet (2017):\\
\textbf{Scientific Workflows for Computational Reproducibility in the
Life Sciences: Status, Challenges and Opportunities}. \emph{Future
Generation Computer Systems} \textbf{75} pp 284--298.\\
\url{https://doi.org/10.1016/j.future.2017.01.012}

\bibitem[6]{6} Björn Grüning, John Chilton, Johannes Köster, Ryan Dale, Nicola
Soranzo, Marius van den Beek, Jeremy Goecks, Rolf Backofen, Anton
Nekrutenko, James Taylor (2018):\\
\textbf{Practical Computational Reproducibility in the Life Sciences}.
\emph{Cell Systems} \textbf{6}(6) pp 631--635.\\
\url{https://doi.org/10.1016/j.cels.2018.03.014}

\bibitem[7]{7} Anna-Lena Lamprecht, Leyla Garcia, Mateusz Kuzak, Carlos
Martinez, Ricardo Arcila, Eva Martin Del Pico, Victoria Dominguez Del
Angel, et al.~\textbf{Towards FAIR Principles for Research Software}.
\emph{Data Science} \textbf{3}(1) pp 37--59.\\
\url{https://doi.org/10.3233/ds-190026}

\bibitem[8]{8} Koenraad De Smedt, Dimitris Koureas, Peter Wittenburg (2020):\\
\textbf{FAIR Digital Objects for Science: From Data Pieces to Actionable
Knowledge Units}.\\
\emph{Publications} \textbf{8}(2):21\\
\url{https://doi.org/10.3390/publications8020021}

\bibitem[9]{9} Carole Goble, Sarah Cohen-Boulakia, Stian Soiland-Reyes, Daniel
Garijo, Yolanda Gil, Michael R. Crusoe, Kristian Peters, Daniel Schober
(2020):\\
\textbf{FAIR Computational Workflows}. \emph{Data Intelligence}
\textbf{2}(1) pp 108--121.\\
\url{https://doi.org/10.1162/dint_a_00033}

\bibitem[10]{10} Pau Andrio, Adam Hospital, Javier Conejero, Luis Jordá, Marc
Del Pino, Laia Codo, Stian Soiland-Reyes, Carole Goble, Daniele Lezzi,
Rosa M. Badia, Modesto Orozco, Josep Ll. Gelpi (2019):\\
\textbf{BioExcel Building Blocks, a software library for interoperable
biomolecular simulation workflows}. \emph{Scientific Data}
\textbf{6}:169\\
\url{https://doi.org/10.1038/s41597-019-0177-4}

\bibitem[11]{11} Jon Ison, Matúš Kalaš, Inge Jonassen, Dan Bolser, Mahmut
Uludag, Hamish McWilliam, James Malone, Rodrigo Lopez, Steve Pettifer,
Peter Rice (2013):\\
\textbf{EDAM: an ontology of bioinformatics operations, types of data
and identifiers, topics and formats}. \emph{Bioinformatics}
\textbf{29}(10) pp 1325--1332.\\
\url{https://doi.org/10.1093/bioinformatics/btt113}

\bibitem[12]{12} Adam Hospital, Genís Bayarri, Stian Soiland-Reyes, Jose Lluis
Gelpi, Pau Andrio, Daniele Lezzi, Sarah Butcher, Ania Niewielska, Yvonne
Westermaier, Rosa Maria Badia, Rodrigo Vargas, Alexandre Bonvin
(2020):\\
\textbf{BioExcel-2 Deliverable 2.3 -- First release of demonstration
workflows.} Project deliverable, \emph{Zenodo}.\\
\url{https://doi.org/10.5281/zenodo.4540432}

\bibitem[13]{13} Thomas Kluyver, Benjamin Ragan-Kelley, Fernando Pérez, Brian
Granger, Matthias Bussonnier, Jonathan Frederic, Kyle Kelley, Jessica
Hamrick, Jason Grout, Sylvain Corlay, Paul Ivanov, Damián Avila, Safia
Abdalla, Carol Willing, Jupyter Development Team (2016):\\
\textbf{Jupyter Notebooks -- a publishing format for reproducible
computational workflows}. \emph{Positioning and Power in Academic
Publishing: Players, Agents and Agendas} pp 87 - 90. Fernando Loizides,
Birgit Schmidt (eds), Proceedings of the 20th International Conference
on Electronic Publishing.\\
\url{https://doi.org/10.3233/978-1-61499-649-1-87}

\bibitem[14]{14} Marijan Beg, Juliette Taka, Thomas Kluyver, Alexander
Konovalov, Min Ragan-Kelley, Nicolas M. Thiery, Hans Fangohr (2021):\\
\textbf{Using Jupyter for Reproducible Scientific Workflows}.
\emph{Computing in Science \& Engineering} \textbf{23}(2) pp 36--46.\\
\url{https://doi.org/10.1109/mcse.2021.3052101}

\bibitem[15]{15} Jupyter Project, Matthias Bussonnier, Jessica Forde, Jeremy
Freeman, Brian Granger, Tim Head, Chris Holdgraf, et al.~(2018):\\
\textbf{Binder 2.0 - Reproducible, Interactive, Sharable Environments
for Science at Scale}. \emph{Proceedings of the 17th Python in Science
Conference}. SciPy, 2018.\\
\url{https://doi.org/10.25080/majora-4af1f417-011}

\bibitem[16]{16} Björn Grüning, Ryan Dale, Andreas Sjödin, Brad A. Chapman,
Jillian Rowe, Christopher H. Tomkins-Tinch, Renan Valieris, the Bioconda
Team, Johannes Köster (2018):\\
\textbf{Bioconda: Sustainable and Comprehensive Software Distribution
for the Life Sciences}. \emph{Nature Methods} \textbf{15} pp 475--476.\\
\url{https://doi.org/10.1038/s41592-018-0046-7}.

\bibitem[17]{17} Ania Niewielska, Sarah Butcher, Yvonne Westermaier (2020):\\
\textbf{BioExcel-2 Deliverable 2.5 - Provision of a Workflow Environment
at BioExcel portal}. \emph{Zenodo}\\
\url{https://doi.org/10.5281/zenodo.4916060}

\bibitem[18]{18} Enis Afgan, Dannon Baker, Bérénice Batut, Marius van den Beek,
Dave Bouvier, Martin Čech, John Chilton, Dave Clements, Nate Coraor,
Björn Grüning, Aysam Guerler, Jennifer Hillman-Jackson, Vahid Jalili,
Helena Rasche, Nicola Soranzo, Jeremy Goecks, James Taylor, Anton
Nekrutenko, and Daniel Blankenberg (2018):\\
\textbf{The Galaxy platform for accessible, reproducible and
collaborative biomedical analyses: 2018 update}. \emph{Nucleic Acids
Research} \textbf{46}\emph{(W1)} pp W537--W544.\\
\url{https://doi.org/10.1093/nar/gky379}

\bibitem[19]{19} Michael R. Crusoe, Sanne Abeln, Alexandru Iosup, Peter Amstutz,
John Chilton, Nebojša Tijanić, Hervé Ménager, Stian Soiland-Reyes,
Carole Goble, The CWL Community (2022):\\
\textbf{Methods Included: Standardizing Computational Reuse and
Portability with the Common Workflow Language}. \emph{Communication of
the ACM} (in press). \emph{arXiv}:2105.07028\\
\url{https://doi.org/10.1145/3486897}

\bibitem[20]{20} Enric Tejedor, Yolanda Becerra, Guillem Alomar, Anna Queralt,
Rosa M. Badia, Jordi Torres, Toni Cortes, Jesús Labarta (2017):\\
\textbf{PyCOMPSs: Parallel computational workflows in Python}. \emph{The
International Journal of High Performance Computing Applications}
\textbf{31}(1) pp 66--82.\\
\url{https://doi.org/10.1177/1094342015594678}

\bibitem[21]{21} Alexander Fillbrunn, Christian Dietz, Julianus Pfeuffer, René
Rahn, Gregory A. Landrum, Michael R. Berthold (2017):\\
\textbf{KNIME for reproducible cross-domain analysis of life science
data}. \emph{Journal of Biotechnology} \textbf{261} pp 149--156.\\
\url{https://doi.org/10.1016/j.jbiotec.2017.07.028}

\bibitem[22]{22} Douglas Lowe (2021):\\
\textbf{Protein MD Setup tutorial using BioExcel Building Blocks (biobb)
in Galaxy}. \emph{WorkflowHub.} Workflow (Galaxy).\\
\url{https://doi.org/10.48546/workflowhub.workflow.194.1}

\bibitem[23]{23} Adam Hospital (2021):\\
\textbf{Protein MD Setup tutorial using BioExcel Building Blocks (biobb)
in KNIME}. \emph{WorkflowHub}. Workflow (KNIME).\\
\url{https://doi.org/10.48546/workflowhub.workflow.201.1}

\bibitem[24]{24} Genís Bayarri, Robin Long (2021):\\
\textbf{Protein MD Setup tutorial using BioExcel Building Blocks (biobb)
in CWL}. \emph{WorkflowHub}. Workflow (CWL).\\
\url{https://doi.org/10.48546/workflowhub.workflow.29.3}

\bibitem[25]{25} Genís Bayarri, Adam Hospital, Douglas Lowe (2021):\\
\textbf{Protein MD Setup tutorial using BioExcel Building Blocks (biobb)
in Jupyter Notebook}. \emph{WorkflowHub}. Workflow (Jupyter Notebook).\\
\url{https://doi.org/10.48546/workflowhub.workflow.120.2}

\bibitem[26]{26} Adam Hospital, Pau Andrio (2021):\\
\textbf{Protein MD Setup HPC tutorial using BioExcel Building Blocks
(biobb) in PyCOMPSs}. \emph{WorkflowHub}. Workflow (PyCOMPSs).\\
\url{https://doi.org/10.48546/workflowhub.workflow.200.1}

\bibitem[27]{27} The BioMoby Consortium (2008):\\
\textbf{Interoperability with Moby 1.0---It's better than sharing your
toothbrush!} \emph{Briefings in Bioinformatics} \textbf{9}(3) pp
220--231.\\
\url{https://doi.org/10.1093/bib/bbn003}

\bibitem[28]{28} Joel Saltz, Scott Oster, Shannon Hastings, Stephen Langella,
Tahsin Kurc, William Sanchez, Manav Kher, Arumani Manisundaram,
Krishnakant Shanbhag, Peter Covitz (2006):\\
\textbf{caGrid: design and implementation of the core architecture of
the cancer biomedical informatics grid}. \emph{Bioinformatics}
\textbf{22}(15) pp 1910--1916.\\
\url{https://doi.org/10.1093/bioinformatics/btl272}

\bibitem[29]{29} Daniel Garijo, Yolanda Gil (2011):\\
\textbf{A New Approach for Publishing Workflows}. \emph{Proceedings of
the 6th Workshop on Workflows in Support of Large-Scale Science - WORKS
'11}.\\
\url{https://doi.org/10.1145/2110497.2110504}

\bibitem[30]{30} Daniel Garijo, Pinar Alper, Khalid Belhajjame, Oscar Corcho,
Yolanda Gil, Carole Goble (2014):\\
\textbf{Common Motifs in Scientific Workflows: An Empirical Analysis}.
\emph{Future Generation Computer Systems} \textbf{36} pp 338--351.\\
\url{https://doi.org/10.1016/j.future.2013.09.018}

\bibitem[31]{31} Renato De Giovanni, Alan R. Williams, Vera Hernández Ernst,
Robert Kulawik, Francisco Quevedo Fernandez, Alex R. Hardisty (2016):\\
\textbf{ENM Components: a new set of web service‐based workflow
components for ecological niche modelling.} \emph{Ecography}
\textbf{39}(4) \textbf{pp} 376--383.\\
\url{https://doi.org/10.1111/ecog.01552}

\bibitem[32]{32} Daniel Blankenberg, Gregory Von Kuster, Emil Bouvier, Dannon
Baker, Enis Afgan, Nicholas Stoler, James Taylor, Anton Nekrutenko, the
Galaxy Team (2014):\\
\textbf{Dissemination of scientific software with Galaxy ToolShed}.
\emph{Genome Biology} \textbf{15}:403\\
\url{https://doi.org/10.1186/gb4161}

\bibitem[33]{33} John Vivian, Arjun Arkal Rao, Frank Austin Nothaft, Christopher
Ketchum, Joel Armstrong, Adam Novak, Jacob Pfeil, Jake Narkizian, Alden
D Deran, Audrey Musselman-Brown, Hannes Schmidt, Peter Amstutz, Brian
Craft, Mary Goldman, Kate Rosenbloom, Melissa Cline, Brian O'Connor,
Megan Hanna, Chet Birger, W James Kent, David A Patterson, Anthony D
Joseph, Jingchun Zhu, Sasha Zaranek, Gad Getz, David Haussler \&
Benedict Paten (2017):\\
\textbf{Toil enables reproducible, open source, big biomedical data
analyses}. \emph{Nature Biotechnology} \textbf{35} pp 314--316.\\
\url{https://doi.org/10.1038/nbt.3772}

\bibitem[34]{34} Stian Soiland-Reyes, Peter Sefton, Mercè Crosas, Leyla Jael
Castro, Frederik Coppens, José M. Fernández, Daniel Garijo, Björn
Grüning, Marco La Rosa, Simone Leo, Eoghan Ó Carragáin, Marc Portier,
Ana Trisovic, RO-Crate Community, Paul Groth, Carole Goble (2022):\\
\textbf{Packaging research artefacts with RO-Crate}. \emph{Data Science}
(pre-press).\\
\url{https://doi.org/10.3233/DS-210053}

\bibitem[35]{35} Jon Ison, Hans Ienasescu, Emil Rydza, Piotr Chmura, Kristoffer
Rapacki, Alban Gaignard, Veit Schwämmle, Jacques van Helden, Matúš
Kalaš, Hervé Ménager (2021):\\
\textbf{biotoolsSchema: a formalized schema for bioinformatics software
description.} \emph{GigaScience}, \textbf{10}(1):giaa157\\
\url{https://doi.org/10.1093/gigascience/giaa157}

\bibitem[36]{36} The CWFR Group, Alex Hardisty (ed.), Peter Wittenburg (ed.)
(2021):\\
\textbf{CWFR Position Paper}. \emph{OSF}, January 6, 2021.\\
\url{https://osf.io/3rekv/}

\bibitem[37]{37} Paul Brack, Peter Crowther, Stian Soiland-Reyes, Stuart Owen,
Douglas Lowe, Alan R Williams, Quentin Groom, Mathias Dillen, Frederik
Coppens, Björn Grüning, Ignacio Eguinoa, Phil Ewels, Carole Goble
(2022):\\
\textbf{10 Simple Rules for making a software tool workflow-ready}.
\emph{PLOS Computational Biology} (accepted), preprint
PCOMPBIOL-D-21-01704R1 at \url{https://zenodo.org/record/5901220}

\bibitem[38]{38} Julie A McMurry, Nick Juty, Niklas Blomberg, Tony Burdett, Tom
Conlin, Nathalie Conte, Mélanie Courtot, John Deck, Michel Dumontier,
Donal K Fellows, Alejandra Gonzalez-Beltran, Philipp Gormanns, Jeffrey
Grethe, Janna Hastings, Jean-Karim Hériché, Henning Hermjakob, Jon C
Ison, Rafael C Jimenez, Simon Jupp, John Kunze, Camille Laibe, Nicolas
Le Novère, James Malone, Maria Jesus Martin, Johanna R McEntyre, Chris
Morris, Juha Muilu, Wolfgang Müller, Philippe Rocca-Serra,
Susanna-Assunta Sansone, Murat Sariyar, Jacky L Snoep, Stian
Soiland-Reyes, Natalie J Stanford, Neil Swainston, Nicole Washington,
Alan R Williams, Sarala M Wimalaratne, Lilly M Winfree, Katherine
Wolstencroft, Carole Goble, Cristopher J Mungall, Melissa A Haendel,
Helen Parkinson (2017):\\
\textbf{Identifiers for the 21st century: How to design, provision, and
reuse identifiers to maximize utility and impact of life science data}.
\emph{PLOS Biology} \textbf{15}(6):e2001414\\
\url{https://doi.org/10.1371/journal.pbio.2001414}

\bibitem[39]{39} Rafael Ferreira da Silva, Henri Casanova, Kyle Chard, Ilkay
Altintas, Rosa M Badia, Bartosz Balis, Tainã Coleman, Frederik Coppens,
Frank Di Natale, Bjoern Enders, Thomas Fahringer, Rosa Filgueira,
Grigori Fursin, Daniel Garijo, Carole Goble, Dorran Howell, Shantenu
Jha, Daniel S. Katz, Daniel Laney, Ulf Leser, Maciej Malawski, Kshitij
Mehta, Loïc Pottier, Jonathan Ozik, J. Luc Peterson, Lavanya
Ramakrishnan, Stian Soiland-Reyes, Douglas Thain, Matthew Wolf (2021):\\
\textbf{A Community Roadmap for Scientific Workflows Research and
Development}.\\
\href{https://arxiv.org/abs/2110.02168}{arXiv:2110.02168} \bibitem[cs.DC]{cs.DC}

\bibitem[40]{40} Leyla Garcia, Erick Antezana, Alexander Garcia, Evan Bolton,
Rafael Jimenez, Pjotr Prins, Juan M. Banda, Toshiaki Katayama (2020):\\
\textbf{Ten simple rules to run a successful BioHackathon}. \emph{PLOS
Computational Biology} \textbf{16}(5):e1007808.\\
\url{https://doi.org/10.1371/journal.pcbi.1007808}



%% Chapter 08 references
\section{References from Chapter 8}

\bibitem[1]{1} Walton S, Livermore L, Bánki O, Cubey RWN, Drinkwater R, Englund
M, Goble C, Groom Q, Kermorvant C, Rey I, Santos CM, Scott B, Williams
AR, Wu Z (2020):\\
Landscape Analysis for the Specimen Data Refinery.\\
Research Ideas and Outcomes 6: e57602.\\
\url{https://doi.org/10.3897/rio.6.e57602}

\bibitem[2]{2} Thiers, B. M., Tulig, M. C., \& Watson, K. A. (2016):\\
Digitization of the New York Botanical Garden herbarium.\\
Brittonia, 68(3), 324-333.\\
\url{https://doi.org/10.1007/s12228-016-9423-7}

\bibitem[3]{3} Nelson, G., Ellis, S., 2019:\\
The history and impact of digitization and digital data mobilization on
biodiversity research.\\
Philosophical Transactions of the Royal Society B: Biological Sciences
374, 20170391.\\
\url{https://doi.org/10.1098/rstb.2017.0391}

\bibitem[4]{4} Nelson, G., Paul, D.L., (2019):\\
DiSSCo, iDigBio and the Future of Global Collaboration.\\
\emph{Biodiversity Information Science and Standards}
\textbf{3}:e37896.\\
\url{https://doi.org/10.3897/biss.3.37896}

\bibitem[5]{5} Addink W, Koureas D, Rubio A. (2019):\\
DiSSCo as a New Regional Model for Scientific Collections in Europe.
\emph{Biodiversity Information Science and Standards}
\textbf{3}:e37502.\\
\url{https://doi.org/10.3897/biss.3.37502}

\bibitem[6]{6} Lannom L, Koureas D, Hardisty AR (2020):\\
\textbf{FAIR Data and Services in Biodiversity Science and
Geoscience}.\\
\emph{Data Intelligence} \textbf{2}(1--2):122--130.\\
\url{https://doi.org/10.1162/dint_a_00034}

\bibitem[7]{7} GBIF Secretariat. (2021). GBIF Science Review 2020.\\
\url{https://doi.org/10.35035/bezp-jj23}

\bibitem[8]{8} Heberling, J. M., Miller, J. T., Noesgaard, D., Weingart, S. B.,
\& Schigel, D. (2021). Data integration enables global biodiversity
synthesis. Proceedings of the National Academy of Sciences, 118(6).\\
\url{https://doi.org/10.1073/pnas.2018093118}

\bibitem[9]{9} Sweeney, P.W., Starly, B., Morris, P.J., Xu, Y., Jones, A.,
Radhakrishnan, S., Grassa, C.J. and Davis, C.C. (2018), Large--scale
digitization of herbarium specimens: Development and usage of an
automated, high--throughput conveyor system. Taxon, 67: 165-178.\\
\url{https://doi.org/10.12705/671.10}

\bibitem[10]{10} Allan, E.L., Livermore, L., Price, B., Shchedrina, O., Smith,
V., (2019). A Novel Automated Mass Digitisation Workflow for Natural
History Microscope Slides. Biodiversity Data Journal 7, e32342.\\
\url{https://doi.org/10.3897/BDJ.7.e32342}

\bibitem[11]{11} Hereld, M., Ferrier, N., (2019). LightningBug ONE: An
experiment in high-throughput digitization of pinned insects.
Biodiversity Information Science and Standards 3, e37228.\\
\url{https://doi.org/10.3897/biss.3.37228}

\bibitem[12]{12} Price, B.W., Dupont, S., Allan, E.L., Blagoderov, V., Butcher,
A.J., Durrant, J., Holtzhausen, P., Kokkini, P., Livermore, L., Hardy,
H., Smith, V., (2018). ALICE: Angled Label Image Capture and Extraction
for high throughput insect specimen digitisation.\\
\url{https://doi.org/10.31219/osf.io/s2p73}

\bibitem[13]{13} Tegelberg, R., Kahanpää, J., Karppinen, J., Mononen, T., Wu,
Z., Saarenmaa, H., (2017). Mass Digitization of Individual Pinned
Insects Using Conveyor-Driven Imaging, in: 2017 IEEE 13th International
Conference on E-Science (e-Science). Presented at the 2017 IEEE 13th
International Conference on e-Science (e-Science), pp.~523--527.\\
\url{https://doi.org/10.1109/eScience.2017.85}

\bibitem[14]{14} Heberling, J.M., Prather, L.A., Tonsor, S.J. (2019) The
Changing Uses of Herbarium Data in an Era of Global Change: An Overview
Using Automated Content Analysis, BioScience, Volume 69, Issue 10,
October 2019, Pages 812--822,\\
\url{https://doi.org/10.1093/biosci/biz094}

\bibitem[15]{15} Kharouba Heather M., Lewthwaite Jayme M. M., Guralnick Rob,
Kerr Jeremy T. and Vellend Mark (2019) Using insect natural history
collections to study global change impacts: challenges and
opportunities. Phil. Trans. R. Soc. B3742017040520170405.\\
\url{https://doi.org/10.1098/rstb.2017.0405}

\bibitem[16]{16} Myrna E Watanabe, The Evolution of Natural History Collections:
New research tools move specimens, data to center stage., BioScience,
Volume 69, Issue 3, March 2019, Pages 163--169,\\
\url{https://doi.org/10.1093/biosci/biy163}

\bibitem[17]{17} Nic Lughadha, E.M., Graziele Staggemeier, V., Vasconcelos,
T.N.C., Walker, B.E., Canteiro, C. and Lucas, E.J. (2019), Harnessing
the potential of integrated systematics for conservation of
taxonomically complex, megadiverse plant groups. Conservation Biology,
33: 511-522.\\
\url{https://doi.org/10.1111/cobi.13289}

\bibitem[18]{18} Owen D, Livermore L, Groom Q, Hardisty A, Leegwater T, van
Walsum M, et al.~(2020). Towards a scientific workflow featuring Natural
Language Processing for the digitisation of natural history collections.
Research Ideas and Outcomes 6: e58030.\\
\url{https://doi.org/10.3897/rio.6.e58030}

\bibitem[19]{19} Jennifer Harrow, John Hancock, ELIXIR-EXCELERATE Community,
Niklas Blomberg (2021): ELIXIR-EXCELERATE: establishing Europe's data
infrastructure for the life science research of the future. \emph{EMBO
Journal} \textbf{40}(6):e107409\\
\url{https://doi.org/10.15252/embj.2020107409}

\bibitem[20]{20} Afgan, E., Baker, D., Batut, B., van den Beek, M., Bouvier, D.,
Čech, M., et al.~(2018). The Galaxy platform for accessible,
reproducible and collaborative biomedical analyses: 2018 update. Nucleic
Acids Research 46, W537--W544.\\
\url{https://doi.org/10.1093/nar/gky379}

\bibitem[21]{21} Crusoe, M.R., Abeln, S., Iosup, A., Amstutz, P., Chilton, J.,
Tijanić, N., et al., (2021). Methods Included: Standardizing
Computational Reuse and Portability with the Common Workflow Language.\\
\textless arXiv:2105.07028\textgreater{} \bibitem[cs]{cs}.

\bibitem[22]{22} Carragáin, E.Ó., Goble, C., Sefton, P., Soiland-Reyes, S.,
(2019). A lightweight approach to research object data packaging.
\emph{Bioinformatics Open Source Conference} (BOSC), ISMB/ECCB 2019,
Basel, Switzerland, 24-25 July 2019 (Session 1, Part 1075).\\
\url{https://doi.org/10.5281/zenodo.3250687}

\bibitem[23]{23} Soiland-Reyes, S., Sefton, P., Crosas, M., Castro, L.J.,
Coppens, F., Fernández, J.M., et al.~(2021). Packaging research
artefacts with RO-Crate. \emph{Data Science} (accepted).\\
\textless arXiv:2108.06503\textgreater{} \bibitem[cs.DL]{cs.DL}.

\bibitem[24]{24} Goble, C., Soiland-Reyes, S., Bacall, F., Owen, S., Williams,
A., Eguinoa, I., et al.~(2021). Implementing FAIR Digital Objects in the
EOSC-Life Workflow Collaboratory. Zenodo.\\
\url{https://doi.org/10.5281/zenodo.4605654}

\bibitem[25]{25} Addink, W., Koureas, D., and Rubio, A. (2019). DiSSCo as a New
Regional Model for Scientific Collections in Europe. Biodiversity
Information Science and Standards, 3, e37502.\\
\url{https://doi.org/10.3897/biss.3.37502}

\bibitem[26]{26} Wilkinson, M.D., Dumontier, M., Aalbersberg, Ij.J., Appleton,
G., Axton, M., Baak, et al.~(2016). The FAIR Guiding Principles for
scientific data management and stewardship. Scientific Data 3, 160018.\\
\url{https://doi.org/10.1038/sdata.2016.18}

\bibitem[27]{27} Wittenburg, P., Hardisty, A., Peer, L., and LeFranc, Y. (2021).
Canonical Workflows to Make Data FAIR. \emph{Data Intelligence} - this
special issue.

\bibitem[28]{28} Hardisty, A. (2019). Provisional Data Management Plan for
DiSSCo infrastructure. Deliverable D6.6. ICEDIG.\\
\url{https://doi.org/10.5281/zenodo.3532937}

\bibitem[29]{29} Koenraad De Smedt, Dimitris Koureas, Peter Wittenburg (2020):\\
\textbf{FAIR Digital Objects for Science: From Data Pieces to Actionable
Knowledge Units}.\\
\emph{Publications} \textbf{8}(2):21\\
\url{https://doi.org/10.3390/publications8020021}

\bibitem[30]{30} Hardisty, A., Saarenmaa, H., Casino, A., Dillen, M., Gödderz,
K., Groom, Q., et al.~(2020). Conceptual design blueprint for the DiSSCo
digitization infrastructure - DELIVERABLE D8.1. Research Ideas and
Outcomes 6: e54280.\\
\url{https://doi.org/10.3897/rio.6.e54280}

\bibitem[31]{31} FDO Coordination Group (2020) FDO Framework.\\
\url{https://github.com/GEDE-RDA-Europe/GEDE/tree/master/FAIR\%20Digital\%20Objects/FDOF}
(accessed August 10, 2021).

\bibitem[32]{32} Triki, A., Bouaziz, B., Mahdi, W., \& Gaikwad, J. (2020).
Objects Detection from Digitized Herbarium Specimen based on Improved
YOLO V3. In VISIGRAPP (4: VISAPP) (pp.~523-529).\\
\url{https://doi.org/10.5220/0009170005230529}

\bibitem[33]{33} Nieva de la Hidalga, A., Rosin, P.L., Sun, X., Livermore, L.,
Durran, J., Turner, J., Dillen, M., Musson, A., Phillips, S., Groom, Q.,
and Hardisty, A.R. (2021) Cross-validation of a semantic segmentation
network for natural history collection specimens. (\emph{in second round
of review at present}).

\bibitem[34]{34} Walton S, Livermore L, Dillen M, De Smedt S, Groom Q, Koivunen
A, Phillips S (2020) A cost analysis of transcription systems. Research
Ideas and Outcomes 6: e56211.\\
\url{https://doi.org/10.3897/rio.6.e56211}

\bibitem[35]{35} Groom, Q., Güntsch, A., Huybrechts, P., Kearney, N., Leachman,
S., Nicolson, N., et al.~(2020). People are essential to linking
biodiversity data, Database 2020.\\
\url{https://doi.org/10.1093/database/baaa072}

\bibitem[36]{36} Knyshov, A., Hoang, S., Weirauch, C., (2021). Pretrained
Convolutional Neural Networks Perform Well in a Challenging Test Case:
Identification of Plant Bugs (Hemiptera: Miridae) Using a Small Number
of Training Images. Insect Systematics and Diversity, Volume 5, Issue 2,
March 2021, 3.\\
\url{https://doi.org/10.1093/isd/ixab004}

\bibitem[37]{37} Hussein, B.R., Malik, O.A., Ong, W.H., Slik, J.W.F.. (2021).
Application of Computer Vision and Machine Learning for Digitized
Herbarium Specimens: A Systematic Literature Review.\\
\url{https://arxiv.org/abs/2104.08732v1}

\bibitem[38]{38} Carranza-Rojas, J., Goeau, H., Bonnet, P. et al.~Going deeper
in the automated identification of Herbarium specimens. BMC Evol Biol
17, 181 (2017).\\
\url{https://doi.org/10.1186/s12862-017-1014-z}

\bibitem[39]{39} Little, D. P., Tulig, M., Tan, K. C., Liu, Y., Belongie, S.,
Kaeser‐Chen, C., Michelangeli, F. A., et al.~2020. An algorithm
competition for automatic species identification from herbarium
specimens. Applications in Plant Sciences 8( 6): e11365.\\
\url{https://doi.org/10.1002/aps3.11365}

\bibitem[40]{40} Pryer, K. M., Tomasi, C., Wang, X., Meineke, E. K., and
Windham, M. D.. 2020. Using computer vision on herbarium specimen images
to discriminate among closely related horsetails (Equisetum).
Applications in Plant Sciences 8( 6): e11372.

\bibitem[41]{41} Unger, J., Merhof, D. \& Renner, S. Computer vision applied to
herbarium specimens of German trees: testing the future utility of the
millions of herbarium specimen images for automated identification. BMC
Evol Biol 16, 248 (2016).\\
\url{https://doi.org/10.1186/s12862-016-0827-5}

\bibitem[42]{42} Atkinson, M., Gesing, S., Montagnat, J., Taylor, I., 2017.
Scientific workflows: Past, present and future. Future Generation
Computer Systems 75, 216--227.\\
\url{https://doi.org/10.1016/j.future.2017.05.041}

\bibitem[43]{43} Peter Amstutz, Maxim Mikheev, Michael R. Crusoe, Nebojša
Tijanić, Samuel Lampa, et al.~(2021): Existing Workflow systems.
\emph{Common Workflow Language wiki}, GitHub.\\
\url{https://s.apache.org/existing-workflow-systems} updated 2021-06-29.
(accessed September 9th, 2021)

\bibitem[44]{44} Hui, Y., (2012). What is a Digital Object? Metaphilosophy 43,
380--395.\\
\url{https://doi.org/10.1111/j.1467-9973.2012.01761.x}

\bibitem[45]{45} Kallinikos, J., Aaltonen, A., Marton, A., (2013). The
Ambivalent Ontology of Digital Artifacts. MIS Quarterly 37, 357--370.
url:\\
\url{https://www.jstor.org/stable/43825913}

\bibitem[46]{46} Kahn, R., Wilensky, R., (2006). A framework for distributed
digital object services. Int J Digit Libr 6, 115--123.\\
\url{https://doi.org/10.1007/s00799-005-0128-x}

\bibitem[47]{47} openDS. (2021). Draft specification for open Digital Specimens
(openDS). url:\\
\url{https://github.com/DiSSCo/openDS}. (accessed August 10, 2021).

\bibitem[48]{48} Bray, T., (2017). The JavaScript Object Notation (JSON) Data
Interchange Format (Request for Comments No.~RFC 8259). Internet
Engineering Task Force.\\
\url{https://doi.org/10.17487/RFC8259}

\bibitem[49]{49} Bechhofer, S., Buchan, I., De Roure, D., Missier, P.,
Ainsworth, J., Bhagat, et al.~(2013). Why linked data is not enough for
scientists. Future Generation Computer Systems, Special section: Recent
advances in e-Science 29, 599--611.\\
\url{https://doi.org/10.1016/j.future.2011.08.004}

\bibitem[50]{50} Kellogg, G., Champin, P.A., Longley, D. (2020). JSON-LD 1.1 A
JSON-based Serialization for Linked Data. W3C Recommendation. Latest
published version:\\
\url{https://www.w3.org/TR/json-ld11/}

\bibitem[51]{51} Schema.org - Schema.org \bibitem[WWW Document]{WWW Document}, n.d. URL\\
\url{https://schema.org/} (accessed August 10, 2021).

\bibitem[52]{52} Corcho, O., González, E., Garijo, D., Palma, R. (2021). D5.1 RO
Model Adapted to EOSC. RELIANCE deliverable, Zenodo.\\
\url{https://doi.org/10.5281/zenodo.4913285}

\bibitem[53]{53} Goble, C., Soiland-Reyes, S., Bacall, F., Owen, S., Williams,
A., Eguinoa, I., et al.~(2021). Implementing FAIR Digital Objects in the
EOSC-Life Workflow Collaboratory. Zenodo. Preprint:\\
\url{https://doi.org/10.5281/zenodo.4605654}

\bibitem[54]{54} Finn Bacall, Alan R Williams, Stian Soiland-Reyes (2021):
Workflow RO-Crate profile 1.0. \emph{WorkflowHub community}\\
\url{https://w3id.org/workflowhub/workflow-ro-crate/1.0} (accessed
November 17th, 2021)

\bibitem[55]{55} Van de Sompel, H., Klein, M., Jones, S., Nelson, M.L., Warner,
S., Devaraju, A., et al.~(2021). FAIR Signposting Profile. 2021-04-20.\\
\url{https://www.signposting.org/FAIR/} (accessed August 10, 2021).

\bibitem[56]{56} Lohonya, K., Livermore, L., Penn, M.G. (2020). Georeferencing
the Natural History Museum's Chinese type collection: of plateaus,
pagodas and plants. Biodiversity Data Journal 8: e50503.\\
\url{https://doi.org/10.3897/BDJ.8.e50503}

\bibitem[57]{57} De Roure, D., and Goble, C. (2010). Anchors in shifting sand:
the primacy of method in the web of data. Web Science Conference 2010,
United States. 26 - 27 Apr 2010. University of Southampton Institutional
Repository\\
\url{https://eprints.soton.ac.uk/270817/} (accessed August 10, 2021).

\bibitem[58]{58} Hardisty, A.R., Bacall, F., Beard, N., Balcázar-Vargas, M.-P.,
Balech, B., Barcza, Z., et al.~(2016). BioVeL: a virtual laboratory for
data analysis and modelling in biodiversity science and ecology. BMC
Ecology 16, 49.\\
\url{https://doi.org/10.1186/s12898-016-0103-y}

\bibitem[59]{59} Dillen, M., Groom, Q., Chagnoux, S., Güntsch, A., Hardisty, A.,
Haston, E., et al.~(2019). A benchmark dataset of herbarium specimen
images with label data. Biodiversity Data Journal 7: e31817.\\
\url{https://doi.org/10.3897/BDJ.7.e31817}

\bibitem[60]{60} Stefan Gössner, Glyn Normington, and Carsten Bormann. 2021.
``JSONPath: Query Expressions for JSON.'' Internet-Draft
draft-ietf-jsonpath-base-02. Internet Engineering Task Force.\\
\url{https://datatracker.ietf.org/doc/html/draft-ietf-jsonpath-base-02}

\bibitem[61]{61} De Roure, D., \& Goble, C. (2010, March). Anchors in Shifting
Sand: The Primacy of Method in the Web of Data.\\
\url{https://eprints.soton.ac.uk/270817/}

\bibitem[62]{62} DONA Foundation. Digital Object Architecture.\\
\url{https://www.dona.net/digitalobjectarchitecture} (accessed August
10, 2021)

\bibitem[63]{63} Digital Object Interface Protocol Specification, version 2.0,
November 2018.\\
\url{https://www.dona.net/sites/default/files/2018-11/DOIPv2Spec_1.pdf}

\bibitem[64]{64} Sun S., Reilly, S., Lannom L., Petrone J. (2003) RFC 3652
Handle System Protocol (ver 2.1) Specification. RFC Editor, USA.\\
\url{https://doi.org/10.17487/RFC3652}

\bibitem[65]{65} Islam, S, Hardisty, A, Addink, W, Weiland, C., and Glöckler, F.
(2020). Incorporating RDA Outputs in the Design of a European Research
Infrastructure for natural history Collections. Data Science Journal,
19: 50, pp.~1--14.\\
\url{https://doi.org/10.5334/dsj-2020-050}

\bibitem[66]{66} Speicher, S., Arwe, J., and Malhotra, A. (2015). Linked Data
Platform 1.0. W3C Recommendation. Latest published version:\\
\url{https://www.w3.org/TR/ldp/}


\end{thebibliography}

